Synthesis failed for dialogflow-python-client-v2

Hello! Autosynth couldn't regenerate dialogflow-python-client-v2. :broken_heart:

Here's the output from running `synth.py`:

```
Cloning into 'working_repo'...
Switched to a new branch 'autosynth'
2020-05-07 05:11:28 [INFO] Running synthtool
2020-05-07 05:11:28 [INFO] ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-07 05:11:28,573 synthtool > Executing /tmpfs/src/github/synthtool/working_repo/synth.py.
On branch autosynth
nothing to commit, working tree clean
2020-05-07 05:11:28,718 synthtool > Ensuring dependencies.
2020-05-07 05:11:28,723 synthtool > Pulling artman image.
latest: Pulling from googleapis/artman
fe703b657a32: Pulling fs layer
f9df1fafd224: Pulling fs layer
a645a4b887f9: Pulling fs layer
57db7fe0b522: Pulling fs layer
ad7df9457e38: Pulling fs layer
5add3b44d39b: Pulling fs layer
9b279b7a7520: Pulling fs layer
a22d25e0a88f: Pulling fs layer
54401ce2cd58: Pulling fs layer
95283fb14fdd: Pulling fs layer
a415f739b308: Pulling fs layer
a9bb3fcb22c8: Pulling fs layer
c63732422ad3: Pulling fs layer
c7fa0aeea839: Pulling fs layer
15773f9a9d11: Pulling fs layer
b070e225a67d: Pulling fs layer
a63805ca0e65: Pulling fs layer
086a99966bf6: Pulling fs layer
ba0755234ac7: Pulling fs layer
c2ed4d189f98: Pulling fs layer
c40e8a94f228: Pulling fs layer
69d13bb66a4e: Pulling fs layer
05cfa4dfe4e0: Pulling fs layer
1e96c343b06a: Pulling fs layer
f057ee848829: Pulling fs layer
d78d453fce70: Pulling fs layer
38512380f3ae: Pulling fs layer
7375eb43508f: Pulling fs layer
5add3b44d39b: Waiting
9b279b7a7520: Waiting
a22d25e0a88f: Waiting
54401ce2cd58: Waiting
95283fb14fdd: Waiting
a415f739b308: Waiting
a9bb3fcb22c8: Waiting
57db7fe0b522: Waiting
c63732422ad3: Waiting
c7fa0aeea839: Waiting
ad7df9457e38: Waiting
15773f9a9d11: Waiting
05cfa4dfe4e0: Waiting
1e96c343b06a: Waiting
f057ee848829: Waiting
b070e225a67d: Waiting
d78d453fce70: Waiting
38512380f3ae: Waiting
a63805ca0e65: Waiting
086a99966bf6: Waiting
7375eb43508f: Waiting
ba0755234ac7: Waiting
69d13bb66a4e: Waiting
c2ed4d189f98: Waiting
f9df1fafd224: Verifying Checksum
f9df1fafd224: Download complete
a645a4b887f9: Verifying Checksum
a645a4b887f9: Download complete
fe703b657a32: Verifying Checksum
fe703b657a32: Download complete
57db7fe0b522: Verifying Checksum
57db7fe0b522: Download complete
ad7df9457e38: Verifying Checksum
ad7df9457e38: Download complete
9b279b7a7520: Verifying Checksum
9b279b7a7520: Download complete
54401ce2cd58: Verifying Checksum
54401ce2cd58: Download complete
a22d25e0a88f: Verifying Checksum
a22d25e0a88f: Download complete
95283fb14fdd: Verifying Checksum
95283fb14fdd: Download complete
fe703b657a32: Pull complete
a415f739b308: Verifying Checksum
a415f739b308: Download complete
f9df1fafd224: Pull complete
a645a4b887f9: Pull complete
57db7fe0b522: Pull complete
a9bb3fcb22c8: Verifying Checksum
a9bb3fcb22c8: Download complete
c7fa0aeea839: Verifying Checksum
c7fa0aeea839: Download complete
ad7df9457e38: Pull complete
15773f9a9d11: Verifying Checksum
15773f9a9d11: Download complete
5add3b44d39b: Verifying Checksum
5add3b44d39b: Download complete
b070e225a67d: Verifying Checksum
b070e225a67d: Download complete
a63805ca0e65: Verifying Checksum
a63805ca0e65: Download complete
086a99966bf6: Verifying Checksum
086a99966bf6: Download complete
c63732422ad3: Verifying Checksum
ba0755234ac7: Verifying Checksum
ba0755234ac7: Download complete
c40e8a94f228: Verifying Checksum
c40e8a94f228: Download complete
05cfa4dfe4e0: Verifying Checksum
05cfa4dfe4e0: Download complete
1e96c343b06a: Verifying Checksum
1e96c343b06a: Download complete
c2ed4d189f98: Download complete
f057ee848829: Download complete
d78d453fce70: Verifying Checksum
d78d453fce70: Download complete
7375eb43508f: Verifying Checksum
7375eb43508f: Download complete
69d13bb66a4e: Verifying Checksum
69d13bb66a4e: Download complete
5add3b44d39b: Pull complete
9b279b7a7520: Pull complete
a22d25e0a88f: Pull complete
54401ce2cd58: Pull complete
95283fb14fdd: Pull complete
a415f739b308: Pull complete
a9bb3fcb22c8: Pull complete
c63732422ad3: Pull complete
c7fa0aeea839: Pull complete
15773f9a9d11: Pull complete
b070e225a67d: Pull complete
a63805ca0e65: Pull complete
086a99966bf6: Pull complete
ba0755234ac7: Pull complete
c2ed4d189f98: Pull complete
c40e8a94f228: Pull complete
69d13bb66a4e: Pull complete
05cfa4dfe4e0: Pull complete
1e96c343b06a: Pull complete
f057ee848829: Pull complete
d78d453fce70: Pull complete
38512380f3ae: Pull complete
7375eb43508f: Pull complete
Digest: sha256:b3b47805231a305d0f40c4bf069df20f6a2635574e6d4259fac651d3f9f6e098
Status: Downloaded newer image for googleapis/artman:latest
2020-05-07 05:12:09,849 synthtool > Cloning googleapis.
2020-05-07 05:12:11,512 synthtool > Running generator for google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-05-07 05:12:40,082 synthtool > Generated code into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2beta1.
2020-05-07 05:12:40,083 synthtool > Placed proto files into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2beta1/google/cloud/dialogflow_v2beta1/proto.
2020-05-07 05:12:40,515 synthtool > No replacements made in ['docs/**/*.rst', 'docs/conf.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-07 05:12:40,516 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2beta1/api.rst.
2020-05-07 05:12:40,516 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2beta1/types.rst.
2020-05-07 05:12:40,526 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2beta1/**/*.py'] for pattern import google.cloud.dialogflow, maybe replacement is no longer needed?
2020-05-07 05:12:40,526 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_contexts_client_v2beta1.py.
2020-05-07 05:12:40,527 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_intents_client_v2beta1.py.
2020-05-07 05:12:40,527 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_entity_types_client_v2beta1.py.
2020-05-07 05:12:40,527 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_session_entity_types_client_v2beta1.py.
2020-05-07 05:12:40,527 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_sessions_client_v2beta1.py.
2020-05-07 05:12:40,527 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_environments_client_v2beta1.py.
2020-05-07 05:12:40,528 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_documents_client_v2beta1.py.
2020-05-07 05:12:40,528 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_knowledge_bases_client_v2beta1.py.
2020-05-07 05:12:40,528 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_agents_client_v2beta1.py.
2020-05-07 05:12:40,529 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/__init__.py.
2020-05-07 05:12:40,529 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/types.py.
2020-05-07 05:12:40,529 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/knowledge_base_pb2_grpc.py.
2020-05-07 05:12:40,530 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_pb2_grpc.py.
2020-05-07 05:12:40,530 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/agent_pb2.py.
2020-05-07 05:12:40,531 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_entity_type_pb2_grpc.py.
2020-05-07 05:12:40,531 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/context_pb2_grpc.py.
2020-05-07 05:12:40,531 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_entity_type_pb2.py.
2020-05-07 05:12:40,531 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/entity_type_pb2_grpc.py.
2020-05-07 05:12:40,532 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/environment_pb2.py.
2020-05-07 05:12:40,532 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/webhook_pb2.py.
2020-05-07 05:12:40,532 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/agent_pb2_grpc.py.
2020-05-07 05:12:40,533 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_pb2.py.
2020-05-07 05:12:40,533 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/intent_pb2_grpc.py.
2020-05-07 05:12:40,533 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/document_pb2_grpc.py.
2020-05-07 05:12:40,533 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/document_pb2.py.
2020-05-07 05:12:40,534 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/intent_pb2.py.
2020-05-07 05:12:40,534 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/environment_pb2_grpc.py.
2020-05-07 05:12:40,535 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-07 05:12:40,535 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-07 05:12:40,536 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-07 05:12:40,536 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-07 05:12:40,537 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-07 05:12:40,537 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-07 05:12:40,537 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-07 05:12:40,537 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-07 05:12:40,538 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-07 05:12:40,538 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/session_entity_types_grpc_transport.py.
2020-05-07 05:12:40,538 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/sessions_grpc_transport.py.
2020-05-07 05:12:40,539 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/agents_grpc_transport.py.
2020-05-07 05:12:40,539 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/knowledge_bases_grpc_transport.py.
2020-05-07 05:12:40,539 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/entity_types_grpc_transport.py.
2020-05-07 05:12:40,539 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/environments_grpc_transport.py.
2020-05-07 05:12:40,539 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/contexts_grpc_transport.py.
2020-05-07 05:12:40,539 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/intents_grpc_transport.py.
2020-05-07 05:12:40,540 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/documents_grpc_transport.py.
2020-05-07 05:12:40,540 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_contexts_client_v2beta1.py.
2020-05-07 05:12:40,540 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_intents_client_v2beta1.py.
2020-05-07 05:12:40,541 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_entity_types_client_v2beta1.py.
2020-05-07 05:12:40,541 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_session_entity_types_client_v2beta1.py.
2020-05-07 05:12:40,541 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_sessions_client_v2beta1.py.
2020-05-07 05:12:40,541 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_environments_client_v2beta1.py.
2020-05-07 05:12:40,541 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_documents_client_v2beta1.py.
2020-05-07 05:12:40,542 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_knowledge_bases_client_v2beta1.py.
2020-05-07 05:12:40,542 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_agents_client_v2beta1.py.
2020-05-07 05:12:40,546 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-07 05:12:40,547 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-07 05:12:40,547 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-07 05:12:40,547 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-07 05:12:40,548 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-07 05:12:40,548 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-07 05:12:40,548 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-07 05:12:40,549 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-07 05:12:40,549 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-07 05:12:40,558 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2beta1/**/*.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-07 05:12:40,563 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-07 05:12:40,564 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-07 05:12:40,564 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-07 05:12:40,564 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-07 05:12:40,565 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-07 05:12:40,565 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-07 05:12:40,565 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-07 05:12:40,566 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-07 05:12:40,566 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-07 05:12:40,572 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-07 05:12:40,573 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-07 05:12:40,573 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-07 05:12:40,574 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-07 05:12:40,574 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-07 05:12:40,574 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-07 05:12:40,575 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-07 05:12:40,575 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-07 05:12:40,575 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-07 05:12:40,580 synthtool > No replacements made in dialogflow_v2beta1/gapic/agents_client.py for pattern (\s+).. raw:: html\s+<pre>curl, maybe replacement is no longer needed?
2020-05-07 05:12:40,580 synthtool > Running generator for google/cloud/dialogflow/v2/artman_dialogflow_v2.yaml.
2020-05-07 05:13:04,503 synthtool > Generated code into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2.
2020-05-07 05:13:04,504 synthtool > Placed proto files into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2/google/cloud/dialogflow_v2/proto.
2020-05-07 05:13:04,921 synthtool > No replacements made in ['docs/**/*.rst', 'docs/conf.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-07 05:13:04,922 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2/api.rst.
2020-05-07 05:13:04,923 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2/types.rst.
2020-05-07 05:13:04,930 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2/**/*.py'] for pattern import google.cloud.dialogflow, maybe replacement is no longer needed?
2020-05-07 05:13:04,931 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_intents_client_v2.py.
2020-05-07 05:13:04,932 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_entity_types_client_v2.py.
2020-05-07 05:13:04,932 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_sessions_client_v2.py.
2020-05-07 05:13:04,932 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_agents_client_v2.py.
2020-05-07 05:13:04,932 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_session_entity_types_client_v2.py.
2020-05-07 05:13:04,933 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_environments_client_v2.py.
2020-05-07 05:13:04,933 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_contexts_client_v2.py.
2020-05-07 05:13:04,933 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/__init__.py.
2020-05-07 05:13:04,933 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/types.py.
2020-05-07 05:13:04,934 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_pb2_grpc.py.
2020-05-07 05:13:04,934 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/agent_pb2.py.
2020-05-07 05:13:04,934 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_entity_type_pb2_grpc.py.
2020-05-07 05:13:04,935 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/context_pb2_grpc.py.
2020-05-07 05:13:04,935 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_entity_type_pb2.py.
2020-05-07 05:13:04,935 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/entity_type_pb2_grpc.py.
2020-05-07 05:13:04,936 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/webhook_pb2.py.
2020-05-07 05:13:04,936 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/agent_pb2_grpc.py.
2020-05-07 05:13:04,937 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_pb2.py.
2020-05-07 05:13:04,937 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/intent_pb2_grpc.py.
2020-05-07 05:13:04,938 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/intent_pb2.py.
2020-05-07 05:13:04,938 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/environment_pb2_grpc.py.
2020-05-07 05:13:04,938 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/contexts_client.py.
2020-05-07 05:13:04,939 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-07 05:13:04,939 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/environments_client.py.
2020-05-07 05:13:04,939 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/sessions_client.py.
2020-05-07 05:13:04,941 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-07 05:13:04,941 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/intents_client.py.
2020-05-07 05:13:04,942 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/agents_client.py.
2020-05-07 05:13:04,942 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/session_entity_types_grpc_transport.py.
2020-05-07 05:13:04,942 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/sessions_grpc_transport.py.
2020-05-07 05:13:04,943 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/agents_grpc_transport.py.
2020-05-07 05:13:04,943 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/entity_types_grpc_transport.py.
2020-05-07 05:13:04,943 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/environments_grpc_transport.py.
2020-05-07 05:13:04,943 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/contexts_grpc_transport.py.
2020-05-07 05:13:04,944 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/intents_grpc_transport.py.
2020-05-07 05:13:04,945 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_intents_client_v2.py.
2020-05-07 05:13:04,945 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_entity_types_client_v2.py.
2020-05-07 05:13:04,945 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_sessions_client_v2.py.
2020-05-07 05:13:04,946 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_agents_client_v2.py.
2020-05-07 05:13:04,946 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_session_entity_types_client_v2.py.
2020-05-07 05:13:04,946 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_environments_client_v2.py.
2020-05-07 05:13:04,946 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_contexts_client_v2.py.
2020-05-07 05:13:04,949 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/contexts_client.py.
2020-05-07 05:13:04,950 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-07 05:13:04,950 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/environments_client.py.
2020-05-07 05:13:04,950 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/sessions_client.py.
2020-05-07 05:13:04,951 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-07 05:13:04,951 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/intents_client.py.
2020-05-07 05:13:04,952 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/agents_client.py.
2020-05-07 05:13:04,957 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/contexts_client.py.
2020-05-07 05:13:04,957 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-07 05:13:04,958 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/environments_client.py.
2020-05-07 05:13:04,958 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/sessions_client.py.
2020-05-07 05:13:04,959 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-07 05:13:04,959 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/intents_client.py.
2020-05-07 05:13:04,960 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/agents_client.py.
2020-05-07 05:13:04,967 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2/**/*.py'] for pattern '-dialogflow', maybe replacement is no longer needed?
2020-05-07 05:13:04,972 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/contexts_client.py.
2020-05-07 05:13:04,972 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-07 05:13:04,973 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/environments_client.py.
2020-05-07 05:13:04,973 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/sessions_client.py.
2020-05-07 05:13:04,974 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-07 05:13:04,974 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/intents_client.py.
2020-05-07 05:13:04,974 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/agents_client.py.
2020-05-07 05:13:04,979 synthtool > No replacements made in dialogflow_v2/gapic/agents_client.py for pattern (\s+).. raw:: html\s+<pre>curl, maybe replacement is no longer needed?
2020-05-07 05:13:04,979 synthtool > Replaced '# Generated by the .*' in dialogflow_v2beta1/proto/session_pb2.py.
2020-05-07 05:13:04,980 synthtool > Replaced '# Generated by the .*' in dialogflow_v2beta1/proto/intent_pb2_grpc.py.
2020-05-07 05:13:04,980 synthtool > Replaced '# Generated by the .*' in dialogflow_v2/proto/intent_pb2_grpc.py.
2020-05-07 05:13:04,980 synthtool > Replaced '# Generated by the .*' in dialogflow_v2/proto/session_pb2.py.
2020-05-07 05:13:04,981 synthtool > No replacements made in ['dialogflow_v2beta1/gapic/intents_client.py', 'dialogflow_v2beta1/gapic/sessions_client.py', 'dialogflow_v2/gapic/intents_client.py'] for pattern # Copyright 2018 Google LLC, maybe replacement is no longer needed?
2020-05-07 05:13:04,982 synthtool > No replacements made in dialogflow_v2/gapic/agents_client.py for pattern }\\\" [\\]\n(\s+retry \(Optional), maybe replacement is no longer needed?
2020-05-07 05:13:04,988 synthtool > No replacements made in dialogflow_v2beta1/proto/intent_pb2.py for pattern \s+-----------, maybe replacement is no longer needed?
2020-05-07 05:13:04,989 synthtool > No replacements made in dialogflow_*/proto/session_pb2.py for pattern ============================================================================, maybe replacement is no longer needed?
2020-05-07 05:13:04,991 synthtool > Replaced '\\s*\\<\\>`__' in dialogflow_v2beta1/proto/audio_config_pb2.py.
2020-05-07 05:13:04,992 synthtool > Replaced '\\s*\\<\\>`__' in dialogflow_v2/proto/audio_config_pb2.py.
2020-05-07 05:13:04,993 synthtool > No replacements made in dialogflow_v2/proto/agent_pb2.py for pattern :math:, maybe replacement is no longer needed?
2020-05-07 05:13:04,993 synthtool > No replacements made in dialogflow_v2/proto/agent_pb2.py for pattern :raw-latex:, maybe replacement is no longer needed?
Error while collecting sessions.
Sessions not found: blacken
2020-05-07 05:13:05,192 synthtool > Failed executing nox -s blacken:

None
2020-05-07 05:13:05,193 synthtool > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/tmpfs/src/github/synthtool/working_repo/synth.py", line 122, in <module>
    s.shell.run(["nox", "-s", "blacken"], hide_output=False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['nox', '-s', 'blacken']' returned non-zero exit status 3.
2020-05-07 05:13:05 [ERROR] Synthesis failed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 584, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 465, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 538, in _inner_main
    deprecated_execution=args.deprecated_execution,
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 115, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](https://sponge/5b75c37f-4751-4c79-af65-77d02a15c8b1).

Autosynth is still having trouble generating dialogflow-python-client-v2. :sob:

Here's the output from running `synth.py`:

```
Cloning into 'working_repo'...
Switched to a new branch 'autosynth'
2020-05-07 14:18:46 [INFO] Running synthtool
2020-05-07 14:18:46 [INFO] ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-07 14:18:46,232 synthtool > Executing /tmpfs/src/github/synthtool/working_repo/synth.py.
On branch autosynth
nothing to commit, working tree clean
2020-05-07 14:18:46,392 synthtool > Ensuring dependencies.
2020-05-07 14:18:46,397 synthtool > Pulling artman image.
latest: Pulling from googleapis/artman
fe703b657a32: Pulling fs layer
f9df1fafd224: Pulling fs layer
a645a4b887f9: Pulling fs layer
57db7fe0b522: Pulling fs layer
ad7df9457e38: Pulling fs layer
5add3b44d39b: Pulling fs layer
9b279b7a7520: Pulling fs layer
a22d25e0a88f: Pulling fs layer
54401ce2cd58: Pulling fs layer
95283fb14fdd: Pulling fs layer
a415f739b308: Pulling fs layer
a9bb3fcb22c8: Pulling fs layer
c63732422ad3: Pulling fs layer
c7fa0aeea839: Pulling fs layer
15773f9a9d11: Pulling fs layer
b070e225a67d: Pulling fs layer
a63805ca0e65: Pulling fs layer
086a99966bf6: Pulling fs layer
ba0755234ac7: Pulling fs layer
c2ed4d189f98: Pulling fs layer
c40e8a94f228: Pulling fs layer
69d13bb66a4e: Pulling fs layer
05cfa4dfe4e0: Pulling fs layer
1e96c343b06a: Pulling fs layer
f057ee848829: Pulling fs layer
d78d453fce70: Pulling fs layer
38512380f3ae: Pulling fs layer
7375eb43508f: Pulling fs layer
15773f9a9d11: Waiting
b070e225a67d: Waiting
a63805ca0e65: Waiting
69d13bb66a4e: Waiting
05cfa4dfe4e0: Waiting
1e96c343b06a: Waiting
f057ee848829: Waiting
d78d453fce70: Waiting
38512380f3ae: Waiting
7375eb43508f: Waiting
086a99966bf6: Waiting
ba0755234ac7: Waiting
c40e8a94f228: Waiting
57db7fe0b522: Waiting
c2ed4d189f98: Waiting
9b279b7a7520: Waiting
a22d25e0a88f: Waiting
54401ce2cd58: Waiting
a415f739b308: Waiting
95283fb14fdd: Waiting
5add3b44d39b: Waiting
ad7df9457e38: Waiting
c63732422ad3: Waiting
c7fa0aeea839: Waiting
f9df1fafd224: Verifying Checksum
f9df1fafd224: Download complete
a645a4b887f9: Verifying Checksum
a645a4b887f9: Download complete
57db7fe0b522: Verifying Checksum
57db7fe0b522: Download complete
fe703b657a32: Verifying Checksum
fe703b657a32: Download complete
ad7df9457e38: Verifying Checksum
ad7df9457e38: Download complete
9b279b7a7520: Verifying Checksum
9b279b7a7520: Download complete
a22d25e0a88f: Verifying Checksum
a22d25e0a88f: Download complete
95283fb14fdd: Verifying Checksum
95283fb14fdd: Download complete
54401ce2cd58: Verifying Checksum
54401ce2cd58: Download complete
a9bb3fcb22c8: Verifying Checksum
a9bb3fcb22c8: Download complete
a415f739b308: Verifying Checksum
a415f739b308: Download complete
c7fa0aeea839: Verifying Checksum
c7fa0aeea839: Download complete
fe703b657a32: Pull complete
5add3b44d39b: Verifying Checksum
5add3b44d39b: Download complete
f9df1fafd224: Pull complete
a645a4b887f9: Pull complete
57db7fe0b522: Pull complete
b070e225a67d: Verifying Checksum
b070e225a67d: Download complete
15773f9a9d11: Verifying Checksum
15773f9a9d11: Download complete
a63805ca0e65: Verifying Checksum
a63805ca0e65: Download complete
086a99966bf6: Verifying Checksum
086a99966bf6: Download complete
c63732422ad3: Verifying Checksum
c63732422ad3: Download complete
ba0755234ac7: Verifying Checksum
ba0755234ac7: Download complete
c40e8a94f228: Verifying Checksum
c40e8a94f228: Download complete
ad7df9457e38: Pull complete
05cfa4dfe4e0: Verifying Checksum
05cfa4dfe4e0: Download complete
1e96c343b06a: Verifying Checksum
1e96c343b06a: Download complete
f057ee848829: Verifying Checksum
f057ee848829: Download complete
c2ed4d189f98: Verifying Checksum
c2ed4d189f98: Download complete
d78d453fce70: Verifying Checksum
d78d453fce70: Download complete
38512380f3ae: Verifying Checksum
38512380f3ae: Download complete
7375eb43508f: Verifying Checksum
7375eb43508f: Download complete
69d13bb66a4e: Verifying Checksum
69d13bb66a4e: Download complete
5add3b44d39b: Pull complete
9b279b7a7520: Pull complete
a22d25e0a88f: Pull complete
54401ce2cd58: Pull complete
95283fb14fdd: Pull complete
a415f739b308: Pull complete
a9bb3fcb22c8: Pull complete
c63732422ad3: Pull complete
c7fa0aeea839: Pull complete
15773f9a9d11: Pull complete
b070e225a67d: Pull complete
a63805ca0e65: Pull complete
086a99966bf6: Pull complete
ba0755234ac7: Pull complete
c2ed4d189f98: Pull complete
c40e8a94f228: Pull complete
69d13bb66a4e: Pull complete
05cfa4dfe4e0: Pull complete
1e96c343b06a: Pull complete
f057ee848829: Pull complete
d78d453fce70: Pull complete
38512380f3ae: Pull complete
7375eb43508f: Pull complete
Digest: sha256:b3b47805231a305d0f40c4bf069df20f6a2635574e6d4259fac651d3f9f6e098
Status: Downloaded newer image for googleapis/artman:latest
2020-05-07 14:19:24,643 synthtool > Cloning googleapis.
2020-05-07 14:19:26,831 synthtool > Running generator for google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-05-07 14:19:58,443 synthtool > Generated code into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2beta1.
2020-05-07 14:19:58,444 synthtool > Placed proto files into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2beta1/google/cloud/dialogflow_v2beta1/proto.
2020-05-07 14:19:58,885 synthtool > No replacements made in ['docs/**/*.rst', 'docs/conf.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-07 14:19:58,887 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2beta1/api.rst.
2020-05-07 14:19:58,887 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2beta1/types.rst.
2020-05-07 14:19:58,896 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2beta1/**/*.py'] for pattern import google.cloud.dialogflow, maybe replacement is no longer needed?
2020-05-07 14:19:58,898 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_knowledge_bases_client_v2beta1.py.
2020-05-07 14:19:58,898 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_contexts_client_v2beta1.py.
2020-05-07 14:19:58,898 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_session_entity_types_client_v2beta1.py.
2020-05-07 14:19:58,898 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_environments_client_v2beta1.py.
2020-05-07 14:19:58,899 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_entity_types_client_v2beta1.py.
2020-05-07 14:19:58,899 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_agents_client_v2beta1.py.
2020-05-07 14:19:58,899 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_intents_client_v2beta1.py.
2020-05-07 14:19:58,899 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_documents_client_v2beta1.py.
2020-05-07 14:19:58,900 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_sessions_client_v2beta1.py.
2020-05-07 14:19:58,900 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/types.py.
2020-05-07 14:19:58,900 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/__init__.py.
2020-05-07 14:19:58,901 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/knowledge_base_pb2_grpc.py.
2020-05-07 14:19:58,901 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_entity_type_pb2_grpc.py.
2020-05-07 14:19:58,901 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/environment_pb2.py.
2020-05-07 14:19:58,901 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/agent_pb2_grpc.py.
2020-05-07 14:19:58,902 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/context_pb2_grpc.py.
2020-05-07 14:19:58,902 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/intent_pb2_grpc.py.
2020-05-07 14:19:58,902 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/document_pb2_grpc.py.
2020-05-07 14:19:58,903 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_pb2.py.
2020-05-07 14:19:58,903 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_pb2_grpc.py.
2020-05-07 14:19:58,903 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/webhook_pb2.py.
2020-05-07 14:19:58,904 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/intent_pb2.py.
2020-05-07 14:19:58,905 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/agent_pb2.py.
2020-05-07 14:19:58,905 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/document_pb2.py.
2020-05-07 14:19:58,905 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/environment_pb2_grpc.py.
2020-05-07 14:19:58,906 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/entity_type_pb2_grpc.py.
2020-05-07 14:19:58,906 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_entity_type_pb2.py.
2020-05-07 14:19:58,907 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-07 14:19:58,907 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-07 14:19:58,907 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-07 14:19:58,908 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-07 14:19:58,908 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-07 14:19:58,908 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-07 14:19:58,909 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-07 14:19:58,909 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-07 14:19:58,910 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-07 14:19:58,910 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/sessions_grpc_transport.py.
2020-05-07 14:19:58,910 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/session_entity_types_grpc_transport.py.
2020-05-07 14:19:58,911 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/agents_grpc_transport.py.
2020-05-07 14:19:58,911 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/intents_grpc_transport.py.
2020-05-07 14:19:58,911 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/knowledge_bases_grpc_transport.py.
2020-05-07 14:19:58,911 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/environments_grpc_transport.py.
2020-05-07 14:19:58,912 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/documents_grpc_transport.py.
2020-05-07 14:19:58,912 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/entity_types_grpc_transport.py.
2020-05-07 14:19:58,912 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/contexts_grpc_transport.py.
2020-05-07 14:19:58,913 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_knowledge_bases_client_v2beta1.py.
2020-05-07 14:19:58,914 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_contexts_client_v2beta1.py.
2020-05-07 14:19:58,914 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_session_entity_types_client_v2beta1.py.
2020-05-07 14:19:58,914 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_environments_client_v2beta1.py.
2020-05-07 14:19:58,914 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_entity_types_client_v2beta1.py.
2020-05-07 14:19:58,915 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_agents_client_v2beta1.py.
2020-05-07 14:19:58,915 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_intents_client_v2beta1.py.
2020-05-07 14:19:58,915 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_documents_client_v2beta1.py.
2020-05-07 14:19:58,915 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_sessions_client_v2beta1.py.
2020-05-07 14:19:58,919 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-07 14:19:58,920 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-07 14:19:58,920 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-07 14:19:58,920 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-07 14:19:58,921 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-07 14:19:58,921 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-07 14:19:58,922 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-07 14:19:58,922 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-07 14:19:58,922 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-07 14:19:58,932 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2beta1/**/*.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-07 14:19:58,937 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-07 14:19:58,937 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-07 14:19:58,938 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-07 14:19:58,938 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-07 14:19:58,938 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-07 14:19:58,939 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-07 14:19:58,939 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-07 14:19:58,939 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-07 14:19:58,940 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-07 14:19:58,947 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-07 14:19:58,947 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-07 14:19:58,948 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-07 14:19:58,948 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-07 14:19:58,948 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-07 14:19:58,949 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-07 14:19:58,949 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-07 14:19:58,949 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-07 14:19:58,950 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-07 14:19:58,955 synthtool > No replacements made in dialogflow_v2beta1/gapic/agents_client.py for pattern (\s+).. raw:: html\s+<pre>curl, maybe replacement is no longer needed?
2020-05-07 14:19:58,956 synthtool > Running generator for google/cloud/dialogflow/v2/artman_dialogflow_v2.yaml.
2020-05-07 14:20:25,043 synthtool > Generated code into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2.
2020-05-07 14:20:25,044 synthtool > Placed proto files into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2/google/cloud/dialogflow_v2/proto.
2020-05-07 14:20:25,467 synthtool > No replacements made in ['docs/**/*.rst', 'docs/conf.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-07 14:20:25,468 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2/api.rst.
2020-05-07 14:20:25,468 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2/types.rst.
2020-05-07 14:20:25,476 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2/**/*.py'] for pattern import google.cloud.dialogflow, maybe replacement is no longer needed?
2020-05-07 14:20:25,476 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_entity_types_client_v2.py.
2020-05-07 14:20:25,477 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_intents_client_v2.py.
2020-05-07 14:20:25,477 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_sessions_client_v2.py.
2020-05-07 14:20:25,477 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_contexts_client_v2.py.
2020-05-07 14:20:25,477 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_environments_client_v2.py.
2020-05-07 14:20:25,477 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_agents_client_v2.py.
2020-05-07 14:20:25,478 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_session_entity_types_client_v2.py.
2020-05-07 14:20:25,479 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/types.py.
2020-05-07 14:20:25,479 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/__init__.py.
2020-05-07 14:20:25,480 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_entity_type_pb2_grpc.py.
2020-05-07 14:20:25,480 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/agent_pb2_grpc.py.
2020-05-07 14:20:25,480 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/context_pb2_grpc.py.
2020-05-07 14:20:25,481 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/intent_pb2_grpc.py.
2020-05-07 14:20:25,481 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_pb2.py.
2020-05-07 14:20:25,481 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_pb2_grpc.py.
2020-05-07 14:20:25,481 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/webhook_pb2.py.
2020-05-07 14:20:25,482 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/intent_pb2.py.
2020-05-07 14:20:25,483 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/agent_pb2.py.
2020-05-07 14:20:25,483 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/environment_pb2_grpc.py.
2020-05-07 14:20:25,483 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/entity_type_pb2_grpc.py.
2020-05-07 14:20:25,484 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_entity_type_pb2.py.
2020-05-07 14:20:25,485 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/sessions_client.py.
2020-05-07 14:20:25,485 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/environments_client.py.
2020-05-07 14:20:25,485 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/contexts_client.py.
2020-05-07 14:20:25,486 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-07 14:20:25,486 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/agents_client.py.
2020-05-07 14:20:25,488 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-07 14:20:25,488 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/intents_client.py.
2020-05-07 14:20:25,489 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/sessions_grpc_transport.py.
2020-05-07 14:20:25,489 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/session_entity_types_grpc_transport.py.
2020-05-07 14:20:25,489 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/agents_grpc_transport.py.
2020-05-07 14:20:25,490 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/intents_grpc_transport.py.
2020-05-07 14:20:25,490 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/environments_grpc_transport.py.
2020-05-07 14:20:25,490 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/entity_types_grpc_transport.py.
2020-05-07 14:20:25,490 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/contexts_grpc_transport.py.
2020-05-07 14:20:25,491 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_entity_types_client_v2.py.
2020-05-07 14:20:25,491 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_intents_client_v2.py.
2020-05-07 14:20:25,491 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_sessions_client_v2.py.
2020-05-07 14:20:25,491 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_contexts_client_v2.py.
2020-05-07 14:20:25,492 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_environments_client_v2.py.
2020-05-07 14:20:25,492 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_agents_client_v2.py.
2020-05-07 14:20:25,492 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_session_entity_types_client_v2.py.
2020-05-07 14:20:25,496 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/sessions_client.py.
2020-05-07 14:20:25,497 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/environments_client.py.
2020-05-07 14:20:25,497 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/contexts_client.py.
2020-05-07 14:20:25,497 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-07 14:20:25,497 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/agents_client.py.
2020-05-07 14:20:25,498 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-07 14:20:25,498 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/intents_client.py.
2020-05-07 14:20:25,504 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/sessions_client.py.
2020-05-07 14:20:25,504 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/environments_client.py.
2020-05-07 14:20:25,504 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/contexts_client.py.
2020-05-07 14:20:25,505 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-07 14:20:25,505 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/agents_client.py.
2020-05-07 14:20:25,505 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-07 14:20:25,506 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/intents_client.py.
2020-05-07 14:20:25,513 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2/**/*.py'] for pattern '-dialogflow', maybe replacement is no longer needed?
2020-05-07 14:20:25,519 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/sessions_client.py.
2020-05-07 14:20:25,519 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/environments_client.py.
2020-05-07 14:20:25,519 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/contexts_client.py.
2020-05-07 14:20:25,520 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-07 14:20:25,520 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/agents_client.py.
2020-05-07 14:20:25,521 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-07 14:20:25,521 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/intents_client.py.
2020-05-07 14:20:25,525 synthtool > No replacements made in dialogflow_v2/gapic/agents_client.py for pattern (\s+).. raw:: html\s+<pre>curl, maybe replacement is no longer needed?
2020-05-07 14:20:25,526 synthtool > Replaced '# Generated by the .*' in dialogflow_v2beta1/proto/session_pb2.py.
2020-05-07 14:20:25,527 synthtool > Replaced '# Generated by the .*' in dialogflow_v2beta1/proto/intent_pb2_grpc.py.
2020-05-07 14:20:25,527 synthtool > Replaced '# Generated by the .*' in dialogflow_v2/proto/intent_pb2_grpc.py.
2020-05-07 14:20:25,527 synthtool > Replaced '# Generated by the .*' in dialogflow_v2/proto/session_pb2.py.
2020-05-07 14:20:25,528 synthtool > No replacements made in ['dialogflow_v2beta1/gapic/intents_client.py', 'dialogflow_v2beta1/gapic/sessions_client.py', 'dialogflow_v2/gapic/intents_client.py'] for pattern # Copyright 2018 Google LLC, maybe replacement is no longer needed?
2020-05-07 14:20:25,529 synthtool > No replacements made in dialogflow_v2/gapic/agents_client.py for pattern }\\\" [\\]\n(\s+retry \(Optional), maybe replacement is no longer needed?
2020-05-07 14:20:25,535 synthtool > No replacements made in dialogflow_v2beta1/proto/intent_pb2.py for pattern \s+-----------, maybe replacement is no longer needed?
2020-05-07 14:20:25,536 synthtool > No replacements made in dialogflow_*/proto/session_pb2.py for pattern ============================================================================, maybe replacement is no longer needed?
2020-05-07 14:20:25,538 synthtool > Replaced '\\s*\\<\\>`__' in dialogflow_v2/proto/audio_config_pb2.py.
2020-05-07 14:20:25,539 synthtool > Replaced '\\s*\\<\\>`__' in dialogflow_v2beta1/proto/audio_config_pb2.py.
2020-05-07 14:20:25,539 synthtool > No replacements made in dialogflow_v2/proto/agent_pb2.py for pattern :math:, maybe replacement is no longer needed?
2020-05-07 14:20:25,540 synthtool > No replacements made in dialogflow_v2/proto/agent_pb2.py for pattern :raw-latex:, maybe replacement is no longer needed?
Error while collecting sessions.
Sessions not found: blacken
2020-05-07 14:20:25,744 synthtool > Failed executing nox -s blacken:

None
2020-05-07 14:20:25,745 synthtool > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/tmpfs/src/github/synthtool/working_repo/synth.py", line 122, in <module>
    s.shell.run(["nox", "-s", "blacken"], hide_output=False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['nox', '-s', 'blacken']' returned non-zero exit status 3.
2020-05-07 14:20:25 [ERROR] Synthesis failed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 584, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 465, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 538, in _inner_main
    deprecated_execution=args.deprecated_execution,
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 115, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](https://sponge/aa65ea1f-6381-42a2-b1da-160265f62da4).

Autosynth is still having trouble generating dialogflow-python-client-v2. :sob:

Here's the output from running `synth.py`:

```
Cloning into 'working_repo'...
Switched to a new branch 'autosynth'
2020-05-07 20:22:26 [INFO] Running synthtool
2020-05-07 20:22:26 [INFO] ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-07 20:22:26,409 synthtool > Executing /tmpfs/src/github/synthtool/working_repo/synth.py.
On branch autosynth
nothing to commit, working tree clean
2020-05-07 20:22:26,571 synthtool > Ensuring dependencies.
2020-05-07 20:22:26,575 synthtool > Pulling artman image.
latest: Pulling from googleapis/artman
fe703b657a32: Pulling fs layer
f9df1fafd224: Pulling fs layer
a645a4b887f9: Pulling fs layer
57db7fe0b522: Pulling fs layer
ad7df9457e38: Pulling fs layer
5add3b44d39b: Pulling fs layer
9b279b7a7520: Pulling fs layer
a22d25e0a88f: Pulling fs layer
54401ce2cd58: Pulling fs layer
95283fb14fdd: Pulling fs layer
a415f739b308: Pulling fs layer
a9bb3fcb22c8: Pulling fs layer
c63732422ad3: Pulling fs layer
c7fa0aeea839: Pulling fs layer
15773f9a9d11: Pulling fs layer
b070e225a67d: Pulling fs layer
a63805ca0e65: Pulling fs layer
086a99966bf6: Pulling fs layer
ba0755234ac7: Pulling fs layer
c2ed4d189f98: Pulling fs layer
c40e8a94f228: Pulling fs layer
69d13bb66a4e: Pulling fs layer
05cfa4dfe4e0: Pulling fs layer
1e96c343b06a: Pulling fs layer
f057ee848829: Pulling fs layer
d78d453fce70: Pulling fs layer
38512380f3ae: Pulling fs layer
7375eb43508f: Pulling fs layer
57db7fe0b522: Waiting
ad7df9457e38: Waiting
5add3b44d39b: Waiting
a9bb3fcb22c8: Waiting
9b279b7a7520: Waiting
c63732422ad3: Waiting
a22d25e0a88f: Waiting
c7fa0aeea839: Waiting
54401ce2cd58: Waiting
15773f9a9d11: Waiting
b070e225a67d: Waiting
95283fb14fdd: Waiting
a415f739b308: Waiting
086a99966bf6: Waiting
a63805ca0e65: Waiting
ba0755234ac7: Waiting
1e96c343b06a: Waiting
c2ed4d189f98: Waiting
38512380f3ae: Waiting
f057ee848829: Waiting
05cfa4dfe4e0: Waiting
d78d453fce70: Waiting
c40e8a94f228: Waiting
7375eb43508f: Waiting
f9df1fafd224: Verifying Checksum
f9df1fafd224: Download complete
57db7fe0b522: Download complete
fe703b657a32: Verifying Checksum
fe703b657a32: Download complete
ad7df9457e38: Verifying Checksum
ad7df9457e38: Download complete
9b279b7a7520: Verifying Checksum
9b279b7a7520: Download complete
a22d25e0a88f: Verifying Checksum
a22d25e0a88f: Download complete
95283fb14fdd: Verifying Checksum
95283fb14fdd: Download complete
54401ce2cd58: Verifying Checksum
54401ce2cd58: Download complete
a9bb3fcb22c8: Verifying Checksum
a9bb3fcb22c8: Download complete
a415f739b308: Verifying Checksum
a415f739b308: Download complete
fe703b657a32: Pull complete
c7fa0aeea839: Verifying Checksum
c7fa0aeea839: Download complete
f9df1fafd224: Pull complete
5add3b44d39b: Verifying Checksum
5add3b44d39b: Download complete
a645a4b887f9: Pull complete
57db7fe0b522: Pull complete
b070e225a67d: Verifying Checksum
b070e225a67d: Download complete
15773f9a9d11: Verifying Checksum
15773f9a9d11: Download complete
a63805ca0e65: Verifying Checksum
a63805ca0e65: Download complete
c63732422ad3: Verifying Checksum
c63732422ad3: Download complete
086a99966bf6: Verifying Checksum
086a99966bf6: Download complete
ba0755234ac7: Download complete
c40e8a94f228: Verifying Checksum
c40e8a94f228: Download complete
ad7df9457e38: Pull complete
05cfa4dfe4e0: Verifying Checksum
05cfa4dfe4e0: Download complete
1e96c343b06a: Verifying Checksum
1e96c343b06a: Download complete
f057ee848829: Verifying Checksum
f057ee848829: Download complete
c2ed4d189f98: Download complete
d78d453fce70: Verifying Checksum
d78d453fce70: Download complete
38512380f3ae: Verifying Checksum
38512380f3ae: Download complete
7375eb43508f: Verifying Checksum
7375eb43508f: Download complete
69d13bb66a4e: Verifying Checksum
69d13bb66a4e: Download complete
5add3b44d39b: Pull complete
9b279b7a7520: Pull complete
a22d25e0a88f: Pull complete
54401ce2cd58: Pull complete
95283fb14fdd: Pull complete
a415f739b308: Pull complete
a9bb3fcb22c8: Pull complete
c63732422ad3: Pull complete
c7fa0aeea839: Pull complete
15773f9a9d11: Pull complete
b070e225a67d: Pull complete
a63805ca0e65: Pull complete
086a99966bf6: Pull complete
ba0755234ac7: Pull complete
c2ed4d189f98: Pull complete
c40e8a94f228: Pull complete
69d13bb66a4e: Pull complete
05cfa4dfe4e0: Pull complete
1e96c343b06a: Pull complete
f057ee848829: Pull complete
d78d453fce70: Pull complete
38512380f3ae: Pull complete
7375eb43508f: Pull complete
Digest: sha256:b3b47805231a305d0f40c4bf069df20f6a2635574e6d4259fac651d3f9f6e098
Status: Downloaded newer image for googleapis/artman:latest
2020-05-07 20:23:04,290 synthtool > Cloning googleapis.
2020-05-07 20:23:06,013 synthtool > Running generator for google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-05-07 20:23:35,761 synthtool > Generated code into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2beta1.
2020-05-07 20:23:35,762 synthtool > Placed proto files into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2beta1/google/cloud/dialogflow_v2beta1/proto.
2020-05-07 20:23:36,194 synthtool > No replacements made in ['docs/**/*.rst', 'docs/conf.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-07 20:23:36,195 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2beta1/types.rst.
2020-05-07 20:23:36,195 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2beta1/api.rst.
2020-05-07 20:23:36,204 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2beta1/**/*.py'] for pattern import google.cloud.dialogflow, maybe replacement is no longer needed?
2020-05-07 20:23:36,206 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_environments_client_v2beta1.py.
2020-05-07 20:23:36,206 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_entity_types_client_v2beta1.py.
2020-05-07 20:23:36,206 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_documents_client_v2beta1.py.
2020-05-07 20:23:36,207 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_knowledge_bases_client_v2beta1.py.
2020-05-07 20:23:36,207 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_intents_client_v2beta1.py.
2020-05-07 20:23:36,207 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_session_entity_types_client_v2beta1.py.
2020-05-07 20:23:36,207 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_contexts_client_v2beta1.py.
2020-05-07 20:23:36,208 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_sessions_client_v2beta1.py.
2020-05-07 20:23:36,208 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_agents_client_v2beta1.py.
2020-05-07 20:23:36,208 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/__init__.py.
2020-05-07 20:23:36,208 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/types.py.
2020-05-07 20:23:36,209 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-07 20:23:36,209 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-07 20:23:36,210 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-07 20:23:36,210 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-07 20:23:36,211 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-07 20:23:36,211 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-07 20:23:36,211 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-07 20:23:36,212 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-07 20:23:36,212 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-07 20:23:36,212 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/contexts_grpc_transport.py.
2020-05-07 20:23:36,213 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/environments_grpc_transport.py.
2020-05-07 20:23:36,213 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/documents_grpc_transport.py.
2020-05-07 20:23:36,213 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/session_entity_types_grpc_transport.py.
2020-05-07 20:23:36,213 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/intents_grpc_transport.py.
2020-05-07 20:23:36,214 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/agents_grpc_transport.py.
2020-05-07 20:23:36,214 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/sessions_grpc_transport.py.
2020-05-07 20:23:36,214 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/entity_types_grpc_transport.py.
2020-05-07 20:23:36,214 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/knowledge_bases_grpc_transport.py.
2020-05-07 20:23:36,215 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/agent_pb2_grpc.py.
2020-05-07 20:23:36,215 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/document_pb2.py.
2020-05-07 20:23:36,215 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/entity_type_pb2_grpc.py.
2020-05-07 20:23:36,216 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/context_pb2_grpc.py.
2020-05-07 20:23:36,216 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/environment_pb2_grpc.py.
2020-05-07 20:23:36,216 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/intent_pb2_grpc.py.
2020-05-07 20:23:36,217 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_pb2_grpc.py.
2020-05-07 20:23:36,217 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/knowledge_base_pb2_grpc.py.
2020-05-07 20:23:36,217 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_entity_type_pb2_grpc.py.
2020-05-07 20:23:36,218 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/environment_pb2.py.
2020-05-07 20:23:36,219 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/intent_pb2.py.
2020-05-07 20:23:36,219 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/document_pb2_grpc.py.
2020-05-07 20:23:36,219 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/webhook_pb2.py.
2020-05-07 20:23:36,220 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/agent_pb2.py.
2020-05-07 20:23:36,220 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_pb2.py.
2020-05-07 20:23:36,221 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_entity_type_pb2.py.
2020-05-07 20:23:36,222 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_environments_client_v2beta1.py.
2020-05-07 20:23:36,222 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_entity_types_client_v2beta1.py.
2020-05-07 20:23:36,222 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_documents_client_v2beta1.py.
2020-05-07 20:23:36,223 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_knowledge_bases_client_v2beta1.py.
2020-05-07 20:23:36,223 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_intents_client_v2beta1.py.
2020-05-07 20:23:36,223 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_session_entity_types_client_v2beta1.py.
2020-05-07 20:23:36,223 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_contexts_client_v2beta1.py.
2020-05-07 20:23:36,224 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_sessions_client_v2beta1.py.
2020-05-07 20:23:36,224 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_agents_client_v2beta1.py.
2020-05-07 20:23:36,225 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-07 20:23:36,225 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-07 20:23:36,226 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-07 20:23:36,226 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-07 20:23:36,226 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-07 20:23:36,227 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-07 20:23:36,227 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-07 20:23:36,227 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-07 20:23:36,228 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-07 20:23:36,240 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2beta1/**/*.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-07 20:23:36,243 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-07 20:23:36,243 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-07 20:23:36,244 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-07 20:23:36,244 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-07 20:23:36,244 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-07 20:23:36,245 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-07 20:23:36,245 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-07 20:23:36,245 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-07 20:23:36,246 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-07 20:23:36,252 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-07 20:23:36,253 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-07 20:23:36,253 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-07 20:23:36,254 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-07 20:23:36,254 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-07 20:23:36,255 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-07 20:23:36,255 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-07 20:23:36,255 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-07 20:23:36,256 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-07 20:23:36,264 synthtool > No replacements made in dialogflow_v2beta1/gapic/agents_client.py for pattern (\s+).. raw:: html\s+<pre>curl, maybe replacement is no longer needed?
2020-05-07 20:23:36,264 synthtool > Running generator for google/cloud/dialogflow/v2/artman_dialogflow_v2.yaml.
2020-05-07 20:24:01,814 synthtool > Generated code into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2.
2020-05-07 20:24:01,815 synthtool > Placed proto files into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2/google/cloud/dialogflow_v2/proto.
2020-05-07 20:24:02,228 synthtool > No replacements made in ['docs/**/*.rst', 'docs/conf.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-07 20:24:02,229 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2/types.rst.
2020-05-07 20:24:02,229 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2/api.rst.
2020-05-07 20:24:02,237 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2/**/*.py'] for pattern import google.cloud.dialogflow, maybe replacement is no longer needed?
2020-05-07 20:24:02,237 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_session_entity_types_client_v2.py.
2020-05-07 20:24:02,238 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_agents_client_v2.py.
2020-05-07 20:24:02,238 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_entity_types_client_v2.py.
2020-05-07 20:24:02,238 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_intents_client_v2.py.
2020-05-07 20:24:02,238 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_environments_client_v2.py.
2020-05-07 20:24:02,239 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_contexts_client_v2.py.
2020-05-07 20:24:02,239 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_sessions_client_v2.py.
2020-05-07 20:24:02,240 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/__init__.py.
2020-05-07 20:24:02,240 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/types.py.
2020-05-07 20:24:02,241 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-07 20:24:02,241 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/agents_client.py.
2020-05-07 20:24:02,242 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/environments_client.py.
2020-05-07 20:24:02,242 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/intents_client.py.
2020-05-07 20:24:02,242 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-07 20:24:02,243 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/contexts_client.py.
2020-05-07 20:24:02,243 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/sessions_client.py.
2020-05-07 20:24:02,244 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/contexts_grpc_transport.py.
2020-05-07 20:24:02,244 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/environments_grpc_transport.py.
2020-05-07 20:24:02,244 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/session_entity_types_grpc_transport.py.
2020-05-07 20:24:02,244 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/intents_grpc_transport.py.
2020-05-07 20:24:02,245 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/agents_grpc_transport.py.
2020-05-07 20:24:02,245 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/sessions_grpc_transport.py.
2020-05-07 20:24:02,245 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/entity_types_grpc_transport.py.
2020-05-07 20:24:02,245 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/agent_pb2_grpc.py.
2020-05-07 20:24:02,246 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/entity_type_pb2_grpc.py.
2020-05-07 20:24:02,246 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/context_pb2_grpc.py.
2020-05-07 20:24:02,246 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/environment_pb2_grpc.py.
2020-05-07 20:24:02,247 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/intent_pb2_grpc.py.
2020-05-07 20:24:02,248 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_pb2_grpc.py.
2020-05-07 20:24:02,248 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_entity_type_pb2_grpc.py.
2020-05-07 20:24:02,249 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/intent_pb2.py.
2020-05-07 20:24:02,250 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/webhook_pb2.py.
2020-05-07 20:24:02,250 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/agent_pb2.py.
2020-05-07 20:24:02,251 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_pb2.py.
2020-05-07 20:24:02,251 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_entity_type_pb2.py.
2020-05-07 20:24:02,251 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_session_entity_types_client_v2.py.
2020-05-07 20:24:02,252 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_agents_client_v2.py.
2020-05-07 20:24:02,252 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_entity_types_client_v2.py.
2020-05-07 20:24:02,252 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_intents_client_v2.py.
2020-05-07 20:24:02,252 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_environments_client_v2.py.
2020-05-07 20:24:02,253 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_contexts_client_v2.py.
2020-05-07 20:24:02,253 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_sessions_client_v2.py.
2020-05-07 20:24:02,255 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-07 20:24:02,255 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/agents_client.py.
2020-05-07 20:24:02,255 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/environments_client.py.
2020-05-07 20:24:02,256 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/intents_client.py.
2020-05-07 20:24:02,256 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-07 20:24:02,256 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/contexts_client.py.
2020-05-07 20:24:02,257 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/sessions_client.py.
2020-05-07 20:24:02,263 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-07 20:24:02,263 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/agents_client.py.
2020-05-07 20:24:02,263 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/environments_client.py.
2020-05-07 20:24:02,264 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/intents_client.py.
2020-05-07 20:24:02,264 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-07 20:24:02,264 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/contexts_client.py.
2020-05-07 20:24:02,265 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/sessions_client.py.
2020-05-07 20:24:02,275 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2/**/*.py'] for pattern '-dialogflow', maybe replacement is no longer needed?
2020-05-07 20:24:02,277 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-07 20:24:02,278 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/agents_client.py.
2020-05-07 20:24:02,278 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/environments_client.py.
2020-05-07 20:24:02,278 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/intents_client.py.
2020-05-07 20:24:02,279 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-07 20:24:02,279 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/contexts_client.py.
2020-05-07 20:24:02,280 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/sessions_client.py.
2020-05-07 20:24:02,286 synthtool > No replacements made in dialogflow_v2/gapic/agents_client.py for pattern (\s+).. raw:: html\s+<pre>curl, maybe replacement is no longer needed?
2020-05-07 20:24:02,287 synthtool > Replaced '# Generated by the .*' in dialogflow_v2beta1/proto/session_pb2.py.
2020-05-07 20:24:02,287 synthtool > Replaced '# Generated by the .*' in dialogflow_v2beta1/proto/intent_pb2_grpc.py.
2020-05-07 20:24:02,287 synthtool > Replaced '# Generated by the .*' in dialogflow_v2/proto/intent_pb2_grpc.py.
2020-05-07 20:24:02,288 synthtool > Replaced '# Generated by the .*' in dialogflow_v2/proto/session_pb2.py.
2020-05-07 20:24:02,289 synthtool > No replacements made in ['dialogflow_v2beta1/gapic/intents_client.py', 'dialogflow_v2beta1/gapic/sessions_client.py', 'dialogflow_v2/gapic/intents_client.py'] for pattern # Copyright 2018 Google LLC, maybe replacement is no longer needed?
2020-05-07 20:24:02,289 synthtool > No replacements made in dialogflow_v2/gapic/agents_client.py for pattern }\\\" [\\]\n(\s+retry \(Optional), maybe replacement is no longer needed?
2020-05-07 20:24:02,295 synthtool > No replacements made in dialogflow_v2beta1/proto/intent_pb2.py for pattern \s+-----------, maybe replacement is no longer needed?
2020-05-07 20:24:02,297 synthtool > No replacements made in dialogflow_*/proto/session_pb2.py for pattern ============================================================================, maybe replacement is no longer needed?
2020-05-07 20:24:02,298 synthtool > Replaced '\\s*\\<\\>`__' in dialogflow_v2/proto/audio_config_pb2.py.
2020-05-07 20:24:02,299 synthtool > Replaced '\\s*\\<\\>`__' in dialogflow_v2beta1/proto/audio_config_pb2.py.
2020-05-07 20:24:02,300 synthtool > No replacements made in dialogflow_v2/proto/agent_pb2.py for pattern :math:, maybe replacement is no longer needed?
2020-05-07 20:24:02,300 synthtool > No replacements made in dialogflow_v2/proto/agent_pb2.py for pattern :raw-latex:, maybe replacement is no longer needed?
Error while collecting sessions.
Sessions not found: blacken
2020-05-07 20:24:02,583 synthtool > Failed executing nox -s blacken:

None
2020-05-07 20:24:02,584 synthtool > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/tmpfs/src/github/synthtool/working_repo/synth.py", line 122, in <module>
    s.shell.run(["nox", "-s", "blacken"], hide_output=False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['nox', '-s', 'blacken']' returned non-zero exit status 3.
2020-05-07 20:24:02 [ERROR] Synthesis failed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 584, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 465, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 538, in _inner_main
    deprecated_execution=args.deprecated_execution,
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 115, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](https://sponge/d164d7bd-44e2-4f88-a28c-71a9dc78c9fd).

Autosynth is still having trouble generating dialogflow-python-client-v2. :sob:

Here's the output from running `synth.py`:

```
Cloning into 'working_repo'...
Switched to a new branch 'autosynth'
2020-05-08 05:11:30 [INFO] Running synthtool
2020-05-08 05:11:30 [INFO] ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-08 05:11:30,452 synthtool > Executing /tmpfs/src/github/synthtool/working_repo/synth.py.
On branch autosynth
nothing to commit, working tree clean
2020-05-08 05:11:30,613 synthtool > Ensuring dependencies.
2020-05-08 05:11:30,618 synthtool > Pulling artman image.
latest: Pulling from googleapis/artman
fe703b657a32: Pulling fs layer
f9df1fafd224: Pulling fs layer
a645a4b887f9: Pulling fs layer
57db7fe0b522: Pulling fs layer
ad7df9457e38: Pulling fs layer
5add3b44d39b: Pulling fs layer
9b279b7a7520: Pulling fs layer
a22d25e0a88f: Pulling fs layer
54401ce2cd58: Pulling fs layer
95283fb14fdd: Pulling fs layer
a415f739b308: Pulling fs layer
a9bb3fcb22c8: Pulling fs layer
c63732422ad3: Pulling fs layer
c7fa0aeea839: Pulling fs layer
15773f9a9d11: Pulling fs layer
b070e225a67d: Pulling fs layer
a63805ca0e65: Pulling fs layer
086a99966bf6: Pulling fs layer
ba0755234ac7: Pulling fs layer
c2ed4d189f98: Pulling fs layer
c40e8a94f228: Pulling fs layer
69d13bb66a4e: Pulling fs layer
05cfa4dfe4e0: Pulling fs layer
1e96c343b06a: Pulling fs layer
f057ee848829: Pulling fs layer
5add3b44d39b: Waiting
d78d453fce70: Pulling fs layer
38512380f3ae: Pulling fs layer
7375eb43508f: Pulling fs layer
9b279b7a7520: Waiting
c7fa0aeea839: Waiting
a22d25e0a88f: Waiting
54401ce2cd58: Waiting
95283fb14fdd: Waiting
15773f9a9d11: Waiting
b070e225a67d: Waiting
a415f739b308: Waiting
a63805ca0e65: Waiting
a9bb3fcb22c8: Waiting
57db7fe0b522: Waiting
086a99966bf6: Waiting
c63732422ad3: Waiting
ad7df9457e38: Waiting
c2ed4d189f98: Waiting
ba0755234ac7: Waiting
f057ee848829: Waiting
7375eb43508f: Waiting
c40e8a94f228: Waiting
d78d453fce70: Waiting
38512380f3ae: Waiting
1e96c343b06a: Waiting
69d13bb66a4e: Waiting
05cfa4dfe4e0: Waiting
a645a4b887f9: Verifying Checksum
a645a4b887f9: Download complete
f9df1fafd224: Download complete
57db7fe0b522: Download complete
fe703b657a32: Verifying Checksum
fe703b657a32: Download complete
ad7df9457e38: Verifying Checksum
ad7df9457e38: Download complete
9b279b7a7520: Verifying Checksum
9b279b7a7520: Download complete
a22d25e0a88f: Verifying Checksum
a22d25e0a88f: Download complete
54401ce2cd58: Verifying Checksum
54401ce2cd58: Download complete
95283fb14fdd: Verifying Checksum
95283fb14fdd: Download complete
a9bb3fcb22c8: Verifying Checksum
a9bb3fcb22c8: Download complete
a415f739b308: Verifying Checksum
a415f739b308: Download complete
c7fa0aeea839: Verifying Checksum
c7fa0aeea839: Download complete
fe703b657a32: Pull complete
f9df1fafd224: Pull complete
5add3b44d39b: Verifying Checksum
5add3b44d39b: Download complete
a645a4b887f9: Pull complete
57db7fe0b522: Pull complete
b070e225a67d: Verifying Checksum
b070e225a67d: Download complete
15773f9a9d11: Verifying Checksum
15773f9a9d11: Download complete
c63732422ad3: Verifying Checksum
c63732422ad3: Download complete
086a99966bf6: Verifying Checksum
086a99966bf6: Download complete
a63805ca0e65: Verifying Checksum
a63805ca0e65: Download complete
ba0755234ac7: Verifying Checksum
ba0755234ac7: Download complete
c40e8a94f228: Verifying Checksum
c40e8a94f228: Download complete
ad7df9457e38: Pull complete
05cfa4dfe4e0: Verifying Checksum
05cfa4dfe4e0: Download complete
1e96c343b06a: Verifying Checksum
1e96c343b06a: Download complete
f057ee848829: Verifying Checksum
f057ee848829: Download complete
c2ed4d189f98: Verifying Checksum
c2ed4d189f98: Download complete
d78d453fce70: Verifying Checksum
d78d453fce70: Download complete
38512380f3ae: Verifying Checksum
38512380f3ae: Download complete
7375eb43508f: Verifying Checksum
7375eb43508f: Download complete
69d13bb66a4e: Verifying Checksum
69d13bb66a4e: Download complete
5add3b44d39b: Pull complete
9b279b7a7520: Pull complete
a22d25e0a88f: Pull complete
54401ce2cd58: Pull complete
95283fb14fdd: Pull complete
a415f739b308: Pull complete
a9bb3fcb22c8: Pull complete
c63732422ad3: Pull complete
c7fa0aeea839: Pull complete
15773f9a9d11: Pull complete
b070e225a67d: Pull complete
a63805ca0e65: Pull complete
086a99966bf6: Pull complete
ba0755234ac7: Pull complete
c2ed4d189f98: Pull complete
c40e8a94f228: Pull complete
69d13bb66a4e: Pull complete
05cfa4dfe4e0: Pull complete
1e96c343b06a: Pull complete
f057ee848829: Pull complete
d78d453fce70: Pull complete
38512380f3ae: Pull complete
7375eb43508f: Pull complete
Digest: sha256:b3b47805231a305d0f40c4bf069df20f6a2635574e6d4259fac651d3f9f6e098
Status: Downloaded newer image for googleapis/artman:latest
2020-05-08 05:12:08,250 synthtool > Cloning googleapis.
2020-05-08 05:12:10,450 synthtool > Running generator for google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-05-08 05:12:40,155 synthtool > Generated code into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2beta1.
2020-05-08 05:12:40,156 synthtool > Placed proto files into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2beta1/google/cloud/dialogflow_v2beta1/proto.
2020-05-08 05:12:40,598 synthtool > No replacements made in ['docs/**/*.rst', 'docs/conf.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-08 05:12:40,599 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2beta1/api.rst.
2020-05-08 05:12:40,599 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2beta1/types.rst.
2020-05-08 05:12:40,609 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2beta1/**/*.py'] for pattern import google.cloud.dialogflow, maybe replacement is no longer needed?
2020-05-08 05:12:40,609 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_environments_client_v2beta1.py.
2020-05-08 05:12:40,610 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_session_entity_types_client_v2beta1.py.
2020-05-08 05:12:40,610 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_agents_client_v2beta1.py.
2020-05-08 05:12:40,610 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_intents_client_v2beta1.py.
2020-05-08 05:12:40,610 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_documents_client_v2beta1.py.
2020-05-08 05:12:40,611 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_knowledge_bases_client_v2beta1.py.
2020-05-08 05:12:40,611 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_sessions_client_v2beta1.py.
2020-05-08 05:12:40,611 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_entity_types_client_v2beta1.py.
2020-05-08 05:12:40,611 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_contexts_client_v2beta1.py.
2020-05-08 05:12:40,612 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/types.py.
2020-05-08 05:12:40,612 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/__init__.py.
2020-05-08 05:12:40,613 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-08 05:12:40,613 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-08 05:12:40,613 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-08 05:12:40,614 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-08 05:12:40,614 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-08 05:12:40,614 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-08 05:12:40,615 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-08 05:12:40,615 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-08 05:12:40,615 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-08 05:12:40,616 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/knowledge_bases_grpc_transport.py.
2020-05-08 05:12:40,616 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/contexts_grpc_transport.py.
2020-05-08 05:12:40,616 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/session_entity_types_grpc_transport.py.
2020-05-08 05:12:40,616 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/sessions_grpc_transport.py.
2020-05-08 05:12:40,617 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/entity_types_grpc_transport.py.
2020-05-08 05:12:40,617 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/agents_grpc_transport.py.
2020-05-08 05:12:40,617 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/intents_grpc_transport.py.
2020-05-08 05:12:40,617 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/environments_grpc_transport.py.
2020-05-08 05:12:40,617 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/documents_grpc_transport.py.
2020-05-08 05:12:40,618 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/environment_pb2.py.
2020-05-08 05:12:40,618 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/document_pb2.py.
2020-05-08 05:12:40,618 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_pb2_grpc.py.
2020-05-08 05:12:40,619 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/intent_pb2_grpc.py.
2020-05-08 05:12:40,619 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/agent_pb2.py.
2020-05-08 05:12:40,619 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_entity_type_pb2_grpc.py.
2020-05-08 05:12:40,619 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/knowledge_base_pb2_grpc.py.
2020-05-08 05:12:40,619 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/webhook_pb2.py.
2020-05-08 05:12:40,620 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/context_pb2_grpc.py.
2020-05-08 05:12:40,620 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/entity_type_pb2_grpc.py.
2020-05-08 05:12:40,620 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_entity_type_pb2.py.
2020-05-08 05:12:40,620 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/environment_pb2_grpc.py.
2020-05-08 05:12:40,621 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/intent_pb2.py.
2020-05-08 05:12:40,622 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_pb2.py.
2020-05-08 05:12:40,623 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/agent_pb2_grpc.py.
2020-05-08 05:12:40,623 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/document_pb2_grpc.py.
2020-05-08 05:12:40,624 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_environments_client_v2beta1.py.
2020-05-08 05:12:40,624 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_session_entity_types_client_v2beta1.py.
2020-05-08 05:12:40,624 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_agents_client_v2beta1.py.
2020-05-08 05:12:40,624 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_intents_client_v2beta1.py.
2020-05-08 05:12:40,625 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_documents_client_v2beta1.py.
2020-05-08 05:12:40,625 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_knowledge_bases_client_v2beta1.py.
2020-05-08 05:12:40,625 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_sessions_client_v2beta1.py.
2020-05-08 05:12:40,625 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_entity_types_client_v2beta1.py.
2020-05-08 05:12:40,626 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_contexts_client_v2beta1.py.
2020-05-08 05:12:40,627 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-08 05:12:40,627 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-08 05:12:40,627 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-08 05:12:40,628 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-08 05:12:40,628 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-08 05:12:40,628 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-08 05:12:40,629 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-08 05:12:40,629 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-08 05:12:40,629 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-08 05:12:40,642 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2beta1/**/*.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-08 05:12:40,644 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-08 05:12:40,644 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-08 05:12:40,645 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-08 05:12:40,645 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-08 05:12:40,645 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-08 05:12:40,646 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-08 05:12:40,646 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-08 05:12:40,646 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-08 05:12:40,647 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-08 05:12:40,653 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-08 05:12:40,654 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-08 05:12:40,654 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-08 05:12:40,654 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-08 05:12:40,655 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-08 05:12:40,655 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-08 05:12:40,656 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-08 05:12:40,656 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-08 05:12:40,656 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-08 05:12:40,664 synthtool > No replacements made in dialogflow_v2beta1/gapic/agents_client.py for pattern (\s+).. raw:: html\s+<pre>curl, maybe replacement is no longer needed?
2020-05-08 05:12:40,664 synthtool > Running generator for google/cloud/dialogflow/v2/artman_dialogflow_v2.yaml.
2020-05-08 05:13:05,494 synthtool > Generated code into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2.
2020-05-08 05:13:05,495 synthtool > Placed proto files into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2/google/cloud/dialogflow_v2/proto.
2020-05-08 05:13:05,913 synthtool > No replacements made in ['docs/**/*.rst', 'docs/conf.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-08 05:13:05,914 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2/api.rst.
2020-05-08 05:13:05,915 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2/types.rst.
2020-05-08 05:13:05,923 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2/**/*.py'] for pattern import google.cloud.dialogflow, maybe replacement is no longer needed?
2020-05-08 05:13:05,924 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_session_entity_types_client_v2.py.
2020-05-08 05:13:05,925 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_entity_types_client_v2.py.
2020-05-08 05:13:05,925 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_agents_client_v2.py.
2020-05-08 05:13:05,925 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_sessions_client_v2.py.
2020-05-08 05:13:05,926 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_environments_client_v2.py.
2020-05-08 05:13:05,926 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_contexts_client_v2.py.
2020-05-08 05:13:05,926 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_intents_client_v2.py.
2020-05-08 05:13:05,927 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/types.py.
2020-05-08 05:13:05,927 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/__init__.py.
2020-05-08 05:13:05,928 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/contexts_client.py.
2020-05-08 05:13:05,928 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/intents_client.py.
2020-05-08 05:13:05,929 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/sessions_client.py.
2020-05-08 05:13:05,929 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-08 05:13:05,930 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/environments_client.py.
2020-05-08 05:13:05,930 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/agents_client.py.
2020-05-08 05:13:05,931 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-08 05:13:05,931 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/contexts_grpc_transport.py.
2020-05-08 05:13:05,931 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/session_entity_types_grpc_transport.py.
2020-05-08 05:13:05,931 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/sessions_grpc_transport.py.
2020-05-08 05:13:05,932 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/entity_types_grpc_transport.py.
2020-05-08 05:13:05,932 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/agents_grpc_transport.py.
2020-05-08 05:13:05,932 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/intents_grpc_transport.py.
2020-05-08 05:13:05,933 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/environments_grpc_transport.py.
2020-05-08 05:13:05,933 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_pb2_grpc.py.
2020-05-08 05:13:05,933 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/intent_pb2_grpc.py.
2020-05-08 05:13:05,934 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/agent_pb2.py.
2020-05-08 05:13:05,934 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_entity_type_pb2_grpc.py.
2020-05-08 05:13:05,935 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/webhook_pb2.py.
2020-05-08 05:13:05,936 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/context_pb2_grpc.py.
2020-05-08 05:13:05,936 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/entity_type_pb2_grpc.py.
2020-05-08 05:13:05,936 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_entity_type_pb2.py.
2020-05-08 05:13:05,937 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/environment_pb2_grpc.py.
2020-05-08 05:13:05,937 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/intent_pb2.py.
2020-05-08 05:13:05,938 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_pb2.py.
2020-05-08 05:13:05,939 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/agent_pb2_grpc.py.
2020-05-08 05:13:05,940 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_session_entity_types_client_v2.py.
2020-05-08 05:13:05,940 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_entity_types_client_v2.py.
2020-05-08 05:13:05,941 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_agents_client_v2.py.
2020-05-08 05:13:05,941 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_sessions_client_v2.py.
2020-05-08 05:13:05,941 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_environments_client_v2.py.
2020-05-08 05:13:05,941 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_contexts_client_v2.py.
2020-05-08 05:13:05,942 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_intents_client_v2.py.
2020-05-08 05:13:05,942 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/contexts_client.py.
2020-05-08 05:13:05,943 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/intents_client.py.
2020-05-08 05:13:05,943 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/sessions_client.py.
2020-05-08 05:13:05,944 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-08 05:13:05,944 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/environments_client.py.
2020-05-08 05:13:05,945 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/agents_client.py.
2020-05-08 05:13:05,945 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-08 05:13:05,950 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/contexts_client.py.
2020-05-08 05:13:05,951 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/intents_client.py.
2020-05-08 05:13:05,951 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/sessions_client.py.
2020-05-08 05:13:05,952 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-08 05:13:05,952 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/environments_client.py.
2020-05-08 05:13:05,953 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/agents_client.py.
2020-05-08 05:13:05,953 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-08 05:13:05,963 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2/**/*.py'] for pattern '-dialogflow', maybe replacement is no longer needed?
2020-05-08 05:13:05,966 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/contexts_client.py.
2020-05-08 05:13:05,966 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/intents_client.py.
2020-05-08 05:13:05,967 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/sessions_client.py.
2020-05-08 05:13:05,967 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-08 05:13:05,968 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/environments_client.py.
2020-05-08 05:13:05,968 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/agents_client.py.
2020-05-08 05:13:05,969 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-08 05:13:05,975 synthtool > No replacements made in dialogflow_v2/gapic/agents_client.py for pattern (\s+).. raw:: html\s+<pre>curl, maybe replacement is no longer needed?
2020-05-08 05:13:05,976 synthtool > Replaced '# Generated by the .*' in dialogflow_v2beta1/proto/session_pb2.py.
2020-05-08 05:13:05,977 synthtool > Replaced '# Generated by the .*' in dialogflow_v2beta1/proto/intent_pb2_grpc.py.
2020-05-08 05:13:05,977 synthtool > Replaced '# Generated by the .*' in dialogflow_v2/proto/intent_pb2_grpc.py.
2020-05-08 05:13:05,977 synthtool > Replaced '# Generated by the .*' in dialogflow_v2/proto/session_pb2.py.
2020-05-08 05:13:05,978 synthtool > No replacements made in ['dialogflow_v2beta1/gapic/intents_client.py', 'dialogflow_v2beta1/gapic/sessions_client.py', 'dialogflow_v2/gapic/intents_client.py'] for pattern # Copyright 2018 Google LLC, maybe replacement is no longer needed?
2020-05-08 05:13:05,979 synthtool > No replacements made in dialogflow_v2/gapic/agents_client.py for pattern }\\\" [\\]\n(\s+retry \(Optional), maybe replacement is no longer needed?
2020-05-08 05:13:05,985 synthtool > No replacements made in dialogflow_v2beta1/proto/intent_pb2.py for pattern \s+-----------, maybe replacement is no longer needed?
2020-05-08 05:13:05,987 synthtool > No replacements made in dialogflow_*/proto/session_pb2.py for pattern ============================================================================, maybe replacement is no longer needed?
2020-05-08 05:13:05,989 synthtool > Replaced '\\s*\\<\\>`__' in dialogflow_v2beta1/proto/audio_config_pb2.py.
2020-05-08 05:13:05,990 synthtool > Replaced '\\s*\\<\\>`__' in dialogflow_v2/proto/audio_config_pb2.py.
2020-05-08 05:13:05,990 synthtool > No replacements made in dialogflow_v2/proto/agent_pb2.py for pattern :math:, maybe replacement is no longer needed?
2020-05-08 05:13:05,991 synthtool > No replacements made in dialogflow_v2/proto/agent_pb2.py for pattern :raw-latex:, maybe replacement is no longer needed?
Error while collecting sessions.
Sessions not found: blacken
2020-05-08 05:13:06,199 synthtool > Failed executing nox -s blacken:

None
2020-05-08 05:13:06,200 synthtool > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/tmpfs/src/github/synthtool/working_repo/synth.py", line 122, in <module>
    s.shell.run(["nox", "-s", "blacken"], hide_output=False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['nox', '-s', 'blacken']' returned non-zero exit status 3.
2020-05-08 05:13:06 [ERROR] Synthesis failed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 584, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 465, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 538, in _inner_main
    deprecated_execution=args.deprecated_execution,
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 115, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](https://sponge/3d13acc5-5b13-4b78-b702-674c200578b7).

Autosynth is still having trouble generating dialogflow-python-client-v2. :sob:

Here's the output from running `synth.py`:

```
2020-05-09 05:11:42 [INFO] logs will be written to: /tmpfs/src/github/synthtool/logs/googleapis/dialogflow-python-client-v2
2020-05-09 05:11:42,274 autosynth > logs will be written to: /tmpfs/src/github/synthtool/logs/googleapis/dialogflow-python-client-v2
Switched to branch 'autosynth'
2020-05-09 05:11:43 [INFO] Running synthtool
2020-05-09 05:11:43,082 autosynth > Running synthtool
2020-05-09 05:11:43 [INFO] ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-09 05:11:43,082 autosynth > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-09 05:11:43,297 synthtool > Executing /home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py.
On branch autosynth
nothing to commit, working tree clean
2020-05-09 05:11:43,466 synthtool > Ensuring dependencies.
2020-05-09 05:11:43,471 synthtool > Pulling artman image.
latest: Pulling from googleapis/artman
fe703b657a32: Pulling fs layer
f9df1fafd224: Pulling fs layer
a645a4b887f9: Pulling fs layer
57db7fe0b522: Pulling fs layer
ad7df9457e38: Pulling fs layer
5add3b44d39b: Pulling fs layer
9b279b7a7520: Pulling fs layer
a22d25e0a88f: Pulling fs layer
54401ce2cd58: Pulling fs layer
95283fb14fdd: Pulling fs layer
a415f739b308: Pulling fs layer
a9bb3fcb22c8: Pulling fs layer
c63732422ad3: Pulling fs layer
c7fa0aeea839: Pulling fs layer
15773f9a9d11: Pulling fs layer
b070e225a67d: Pulling fs layer
a63805ca0e65: Pulling fs layer
086a99966bf6: Pulling fs layer
ba0755234ac7: Pulling fs layer
c2ed4d189f98: Pulling fs layer
c40e8a94f228: Pulling fs layer
69d13bb66a4e: Pulling fs layer
05cfa4dfe4e0: Pulling fs layer
1e96c343b06a: Pulling fs layer
f057ee848829: Pulling fs layer
d78d453fce70: Pulling fs layer
38512380f3ae: Pulling fs layer
7375eb43508f: Pulling fs layer
c40e8a94f228: Waiting
69d13bb66a4e: Waiting
9b279b7a7520: Waiting
05cfa4dfe4e0: Waiting
c7fa0aeea839: Waiting
1e96c343b06a: Waiting
f057ee848829: Waiting
a22d25e0a88f: Waiting
54401ce2cd58: Waiting
d78d453fce70: Waiting
95283fb14fdd: Waiting
38512380f3ae: Waiting
7375eb43508f: Waiting
a415f739b308: Waiting
a9bb3fcb22c8: Waiting
b070e225a67d: Waiting
a63805ca0e65: Waiting
086a99966bf6: Waiting
ba0755234ac7: Waiting
57db7fe0b522: Waiting
ad7df9457e38: Waiting
5add3b44d39b: Waiting
c2ed4d189f98: Waiting
15773f9a9d11: Waiting
a645a4b887f9: Verifying Checksum
a645a4b887f9: Download complete
f9df1fafd224: Verifying Checksum
f9df1fafd224: Download complete
57db7fe0b522: Verifying Checksum
57db7fe0b522: Download complete
fe703b657a32: Verifying Checksum
fe703b657a32: Download complete
ad7df9457e38: Verifying Checksum
ad7df9457e38: Download complete
9b279b7a7520: Verifying Checksum
9b279b7a7520: Download complete
a22d25e0a88f: Verifying Checksum
a22d25e0a88f: Download complete
95283fb14fdd: Verifying Checksum
95283fb14fdd: Download complete
54401ce2cd58: Verifying Checksum
54401ce2cd58: Download complete
a9bb3fcb22c8: Verifying Checksum
a9bb3fcb22c8: Download complete
a415f739b308: Verifying Checksum
a415f739b308: Download complete
c7fa0aeea839: Verifying Checksum
c7fa0aeea839: Download complete
fe703b657a32: Pull complete
5add3b44d39b: Verifying Checksum
5add3b44d39b: Download complete
f9df1fafd224: Pull complete
a645a4b887f9: Pull complete
57db7fe0b522: Pull complete
b070e225a67d: Verifying Checksum
b070e225a67d: Download complete
15773f9a9d11: Download complete
c63732422ad3: Verifying Checksum
c63732422ad3: Download complete
086a99966bf6: Verifying Checksum
086a99966bf6: Download complete
a63805ca0e65: Verifying Checksum
a63805ca0e65: Download complete
ba0755234ac7: Verifying Checksum
ba0755234ac7: Download complete
c40e8a94f228: Verifying Checksum
c40e8a94f228: Download complete
05cfa4dfe4e0: Verifying Checksum
05cfa4dfe4e0: Download complete
ad7df9457e38: Pull complete
1e96c343b06a: Verifying Checksum
1e96c343b06a: Download complete
f057ee848829: Verifying Checksum
f057ee848829: Download complete
c2ed4d189f98: Verifying Checksum
c2ed4d189f98: Download complete
d78d453fce70: Verifying Checksum
d78d453fce70: Download complete
38512380f3ae: Verifying Checksum
38512380f3ae: Download complete
7375eb43508f: Verifying Checksum
7375eb43508f: Download complete
69d13bb66a4e: Verifying Checksum
69d13bb66a4e: Download complete
5add3b44d39b: Pull complete
9b279b7a7520: Pull complete
a22d25e0a88f: Pull complete
54401ce2cd58: Pull complete
95283fb14fdd: Pull complete
a415f739b308: Pull complete
a9bb3fcb22c8: Pull complete
c63732422ad3: Pull complete
c7fa0aeea839: Pull complete
15773f9a9d11: Pull complete
b070e225a67d: Pull complete
a63805ca0e65: Pull complete
086a99966bf6: Pull complete
ba0755234ac7: Pull complete
c2ed4d189f98: Pull complete
c40e8a94f228: Pull complete
69d13bb66a4e: Pull complete
05cfa4dfe4e0: Pull complete
1e96c343b06a: Pull complete
f057ee848829: Pull complete
d78d453fce70: Pull complete
38512380f3ae: Pull complete
7375eb43508f: Pull complete
Digest: sha256:b3b47805231a305d0f40c4bf069df20f6a2635574e6d4259fac651d3f9f6e098
Status: Downloaded newer image for googleapis/artman:latest
2020-05-09 05:12:21,895 synthtool > Cloning googleapis.
2020-05-09 05:12:23,598 synthtool > Running generator for google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-05-09 05:12:57,268 synthtool > Generated code into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2beta1.
2020-05-09 05:12:57,269 synthtool > Placed proto files into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2beta1/google/cloud/dialogflow_v2beta1/proto.
2020-05-09 05:12:57,709 synthtool > No replacements made in ['docs/**/*.rst', 'docs/conf.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-09 05:12:57,711 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2beta1/api.rst.
2020-05-09 05:12:57,711 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2beta1/types.rst.
2020-05-09 05:12:57,721 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2beta1/**/*.py'] for pattern import google.cloud.dialogflow, maybe replacement is no longer needed?
2020-05-09 05:12:57,723 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_session_entity_types_client_v2beta1.py.
2020-05-09 05:12:57,723 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_agents_client_v2beta1.py.
2020-05-09 05:12:57,724 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_environments_client_v2beta1.py.
2020-05-09 05:12:57,724 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_entity_types_client_v2beta1.py.
2020-05-09 05:12:57,724 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_intents_client_v2beta1.py.
2020-05-09 05:12:57,724 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_knowledge_bases_client_v2beta1.py.
2020-05-09 05:12:57,725 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_sessions_client_v2beta1.py.
2020-05-09 05:12:57,725 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_contexts_client_v2beta1.py.
2020-05-09 05:12:57,725 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_documents_client_v2beta1.py.
2020-05-09 05:12:57,726 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/types.py.
2020-05-09 05:12:57,726 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/__init__.py.
2020-05-09 05:12:57,726 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_entity_type_pb2_grpc.py.
2020-05-09 05:12:57,727 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_entity_type_pb2.py.
2020-05-09 05:12:57,727 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/environment_pb2.py.
2020-05-09 05:12:57,728 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/intent_pb2.py.
2020-05-09 05:12:57,728 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/intent_pb2_grpc.py.
2020-05-09 05:12:57,729 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/webhook_pb2.py.
2020-05-09 05:12:57,729 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/entity_type_pb2_grpc.py.
2020-05-09 05:12:57,729 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/document_pb2.py.
2020-05-09 05:12:57,730 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/context_pb2_grpc.py.
2020-05-09 05:12:57,730 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/agent_pb2_grpc.py.
2020-05-09 05:12:57,731 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/document_pb2_grpc.py.
2020-05-09 05:12:57,731 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/agent_pb2.py.
2020-05-09 05:12:57,732 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_pb2.py.
2020-05-09 05:12:57,732 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/knowledge_base_pb2_grpc.py.
2020-05-09 05:12:57,732 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_pb2_grpc.py.
2020-05-09 05:12:57,733 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/environment_pb2_grpc.py.
2020-05-09 05:12:57,733 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-09 05:12:57,733 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-09 05:12:57,734 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-09 05:12:57,734 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-09 05:12:57,734 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-09 05:12:57,735 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-09 05:12:57,736 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-09 05:12:57,736 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-09 05:12:57,736 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-09 05:12:57,737 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/knowledge_bases_grpc_transport.py.
2020-05-09 05:12:57,737 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/sessions_grpc_transport.py.
2020-05-09 05:12:57,737 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/environments_grpc_transport.py.
2020-05-09 05:12:57,737 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/session_entity_types_grpc_transport.py.
2020-05-09 05:12:57,738 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/contexts_grpc_transport.py.
2020-05-09 05:12:57,738 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/intents_grpc_transport.py.
2020-05-09 05:12:57,738 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/entity_types_grpc_transport.py.
2020-05-09 05:12:57,738 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/agents_grpc_transport.py.
2020-05-09 05:12:57,739 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/documents_grpc_transport.py.
2020-05-09 05:12:57,740 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_session_entity_types_client_v2beta1.py.
2020-05-09 05:12:57,740 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_agents_client_v2beta1.py.
2020-05-09 05:12:57,741 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_environments_client_v2beta1.py.
2020-05-09 05:12:57,741 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_entity_types_client_v2beta1.py.
2020-05-09 05:12:57,741 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_intents_client_v2beta1.py.
2020-05-09 05:12:57,741 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_knowledge_bases_client_v2beta1.py.
2020-05-09 05:12:57,742 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_sessions_client_v2beta1.py.
2020-05-09 05:12:57,742 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_contexts_client_v2beta1.py.
2020-05-09 05:12:57,742 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_documents_client_v2beta1.py.
2020-05-09 05:12:57,746 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-09 05:12:57,747 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-09 05:12:57,747 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-09 05:12:57,747 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-09 05:12:57,748 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-09 05:12:57,749 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-09 05:12:57,749 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-09 05:12:57,749 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-09 05:12:57,750 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-09 05:12:57,759 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2beta1/**/*.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-09 05:12:57,764 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-09 05:12:57,765 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-09 05:12:57,765 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-09 05:12:57,766 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-09 05:12:57,766 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-09 05:12:57,767 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-09 05:12:57,767 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-09 05:12:57,768 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-09 05:12:57,768 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-09 05:12:57,775 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-09 05:12:57,776 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-09 05:12:57,776 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-09 05:12:57,776 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-09 05:12:57,777 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-09 05:12:57,778 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-09 05:12:57,778 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-09 05:12:57,779 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-09 05:12:57,779 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-09 05:12:57,784 synthtool > No replacements made in dialogflow_v2beta1/gapic/agents_client.py for pattern (\s+).. raw:: html\s+<pre>curl, maybe replacement is no longer needed?
2020-05-09 05:12:57,784 synthtool > Running generator for google/cloud/dialogflow/v2/artman_dialogflow_v2.yaml.
2020-05-09 05:13:25,968 synthtool > Generated code into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2.
2020-05-09 05:13:25,969 synthtool > Placed proto files into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2/google/cloud/dialogflow_v2/proto.
2020-05-09 05:13:26,397 synthtool > No replacements made in ['docs/**/*.rst', 'docs/conf.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-09 05:13:26,398 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2/api.rst.
2020-05-09 05:13:26,398 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2/types.rst.
2020-05-09 05:13:26,406 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2/**/*.py'] for pattern import google.cloud.dialogflow, maybe replacement is no longer needed?
2020-05-09 05:13:26,407 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_sessions_client_v2.py.
2020-05-09 05:13:26,407 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_entity_types_client_v2.py.
2020-05-09 05:13:26,407 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_contexts_client_v2.py.
2020-05-09 05:13:26,407 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_session_entity_types_client_v2.py.
2020-05-09 05:13:26,408 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_intents_client_v2.py.
2020-05-09 05:13:26,408 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_agents_client_v2.py.
2020-05-09 05:13:26,408 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_environments_client_v2.py.
2020-05-09 05:13:26,410 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/types.py.
2020-05-09 05:13:26,410 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/__init__.py.
2020-05-09 05:13:26,410 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_entity_type_pb2_grpc.py.
2020-05-09 05:13:26,410 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_entity_type_pb2.py.
2020-05-09 05:13:26,411 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/intent_pb2.py.
2020-05-09 05:13:26,412 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/intent_pb2_grpc.py.
2020-05-09 05:13:26,412 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/webhook_pb2.py.
2020-05-09 05:13:26,412 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/entity_type_pb2_grpc.py.
2020-05-09 05:13:26,413 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/context_pb2_grpc.py.
2020-05-09 05:13:26,413 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/agent_pb2_grpc.py.
2020-05-09 05:13:26,414 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/agent_pb2.py.
2020-05-09 05:13:26,414 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_pb2.py.
2020-05-09 05:13:26,414 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_pb2_grpc.py.
2020-05-09 05:13:26,415 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/environment_pb2_grpc.py.
2020-05-09 05:13:26,415 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/agents_client.py.
2020-05-09 05:13:26,415 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/intents_client.py.
2020-05-09 05:13:26,416 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-09 05:13:26,416 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-09 05:13:26,418 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/environments_client.py.
2020-05-09 05:13:26,418 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/sessions_client.py.
2020-05-09 05:13:26,419 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/contexts_client.py.
2020-05-09 05:13:26,419 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/sessions_grpc_transport.py.
2020-05-09 05:13:26,419 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/environments_grpc_transport.py.
2020-05-09 05:13:26,420 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/session_entity_types_grpc_transport.py.
2020-05-09 05:13:26,420 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/contexts_grpc_transport.py.
2020-05-09 05:13:26,420 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/intents_grpc_transport.py.
2020-05-09 05:13:26,420 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/entity_types_grpc_transport.py.
2020-05-09 05:13:26,421 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/agents_grpc_transport.py.
2020-05-09 05:13:26,421 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_sessions_client_v2.py.
2020-05-09 05:13:26,421 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_entity_types_client_v2.py.
2020-05-09 05:13:26,422 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_contexts_client_v2.py.
2020-05-09 05:13:26,422 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_session_entity_types_client_v2.py.
2020-05-09 05:13:26,422 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_intents_client_v2.py.
2020-05-09 05:13:26,422 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_agents_client_v2.py.
2020-05-09 05:13:26,423 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_environments_client_v2.py.
2020-05-09 05:13:26,427 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/agents_client.py.
2020-05-09 05:13:26,427 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/intents_client.py.
2020-05-09 05:13:26,427 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-09 05:13:26,427 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-09 05:13:26,428 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/environments_client.py.
2020-05-09 05:13:26,429 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/sessions_client.py.
2020-05-09 05:13:26,429 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/contexts_client.py.
2020-05-09 05:13:26,434 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/agents_client.py.
2020-05-09 05:13:26,435 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/intents_client.py.
2020-05-09 05:13:26,435 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-09 05:13:26,435 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-09 05:13:26,436 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/environments_client.py.
2020-05-09 05:13:26,436 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/sessions_client.py.
2020-05-09 05:13:26,437 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/contexts_client.py.
2020-05-09 05:13:26,444 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2/**/*.py'] for pattern '-dialogflow', maybe replacement is no longer needed?
2020-05-09 05:13:26,449 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/agents_client.py.
2020-05-09 05:13:26,450 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/intents_client.py.
2020-05-09 05:13:26,450 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-09 05:13:26,450 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-09 05:13:26,451 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/environments_client.py.
2020-05-09 05:13:26,451 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/sessions_client.py.
2020-05-09 05:13:26,452 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/contexts_client.py.
2020-05-09 05:13:26,457 synthtool > No replacements made in dialogflow_v2/gapic/agents_client.py for pattern (\s+).. raw:: html\s+<pre>curl, maybe replacement is no longer needed?
2020-05-09 05:13:26,458 synthtool > Replaced '# Generated by the .*' in dialogflow_v2beta1/proto/session_pb2.py.
2020-05-09 05:13:26,458 synthtool > Replaced '# Generated by the .*' in dialogflow_v2beta1/proto/intent_pb2_grpc.py.
2020-05-09 05:13:26,458 synthtool > Replaced '# Generated by the .*' in dialogflow_v2/proto/intent_pb2_grpc.py.
2020-05-09 05:13:26,459 synthtool > Replaced '# Generated by the .*' in dialogflow_v2/proto/session_pb2.py.
2020-05-09 05:13:26,460 synthtool > No replacements made in ['dialogflow_v2beta1/gapic/intents_client.py', 'dialogflow_v2beta1/gapic/sessions_client.py', 'dialogflow_v2/gapic/intents_client.py'] for pattern # Copyright 2018 Google LLC, maybe replacement is no longer needed?
2020-05-09 05:13:26,460 synthtool > No replacements made in dialogflow_v2/gapic/agents_client.py for pattern }\\\" [\\]\n(\s+retry \(Optional), maybe replacement is no longer needed?
2020-05-09 05:13:26,466 synthtool > No replacements made in dialogflow_v2beta1/proto/intent_pb2.py for pattern \s+-----------, maybe replacement is no longer needed?
2020-05-09 05:13:26,468 synthtool > No replacements made in dialogflow_*/proto/session_pb2.py for pattern ============================================================================, maybe replacement is no longer needed?
2020-05-09 05:13:26,470 synthtool > Replaced '\\s*\\<\\>`__' in dialogflow_v2beta1/proto/audio_config_pb2.py.
2020-05-09 05:13:26,471 synthtool > Replaced '\\s*\\<\\>`__' in dialogflow_v2/proto/audio_config_pb2.py.
2020-05-09 05:13:26,471 synthtool > No replacements made in dialogflow_v2/proto/agent_pb2.py for pattern :math:, maybe replacement is no longer needed?
2020-05-09 05:13:26,472 synthtool > No replacements made in dialogflow_v2/proto/agent_pb2.py for pattern :raw-latex:, maybe replacement is no longer needed?
Error while collecting sessions.
Sessions not found: blacken
2020-05-09 05:13:26,697 synthtool > Failed executing nox -s blacken:

None
2020-05-09 05:13:26,698 synthtool > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py", line 122, in <module>
    s.shell.run(["nox", "-s", "blacken"], hide_output=False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['nox', '-s', 'blacken']' returned non-zero exit status 3.
2020-05-09 05:13:26 [ERROR] Synthesis failed
2020-05-09 05:13:26,755 autosynth > Synthesis failed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 599, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 471, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 549, in _inner_main
    ).synthesize(base_synth_log_path)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 118, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](https://sponge/bb436efe-87a1-4af8-a237-e1032e9e4b11).

Autosynth is still having trouble generating dialogflow-python-client-v2. :sob:

Here's the output from running `synth.py`:

```
2020-05-10 05:11:28 [INFO] logs will be written to: /tmpfs/src/github/synthtool/logs/googleapis/dialogflow-python-client-v2
2020-05-10 05:11:28,814 autosynth > logs will be written to: /tmpfs/src/github/synthtool/logs/googleapis/dialogflow-python-client-v2
Switched to branch 'autosynth'
2020-05-10 05:11:29 [INFO] Running synthtool
2020-05-10 05:11:29,626 autosynth > Running synthtool
2020-05-10 05:11:29 [INFO] ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-10 05:11:29,626 autosynth > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-10 05:11:29,846 synthtool > Executing /home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py.
On branch autosynth
nothing to commit, working tree clean
2020-05-10 05:11:30,005 synthtool > Ensuring dependencies.
2020-05-10 05:11:30,009 synthtool > Pulling artman image.
latest: Pulling from googleapis/artman
fe703b657a32: Pulling fs layer
f9df1fafd224: Pulling fs layer
a645a4b887f9: Pulling fs layer
57db7fe0b522: Pulling fs layer
ad7df9457e38: Pulling fs layer
5add3b44d39b: Pulling fs layer
9b279b7a7520: Pulling fs layer
a22d25e0a88f: Pulling fs layer
54401ce2cd58: Pulling fs layer
95283fb14fdd: Pulling fs layer
a415f739b308: Pulling fs layer
a9bb3fcb22c8: Pulling fs layer
c63732422ad3: Pulling fs layer
c7fa0aeea839: Pulling fs layer
15773f9a9d11: Pulling fs layer
b070e225a67d: Pulling fs layer
57db7fe0b522: Waiting
a63805ca0e65: Pulling fs layer
086a99966bf6: Pulling fs layer
ba0755234ac7: Pulling fs layer
5add3b44d39b: Waiting
c2ed4d189f98: Pulling fs layer
a415f739b308: Waiting
c40e8a94f228: Pulling fs layer
9b279b7a7520: Waiting
69d13bb66a4e: Pulling fs layer
ad7df9457e38: Waiting
a9bb3fcb22c8: Waiting
c63732422ad3: Waiting
05cfa4dfe4e0: Pulling fs layer
a22d25e0a88f: Waiting
1e96c343b06a: Pulling fs layer
95283fb14fdd: Waiting
54401ce2cd58: Waiting
f057ee848829: Pulling fs layer
c7fa0aeea839: Waiting
d78d453fce70: Pulling fs layer
15773f9a9d11: Waiting
b070e225a67d: Waiting
ba0755234ac7: Waiting
a63805ca0e65: Waiting
c2ed4d189f98: Waiting
086a99966bf6: Waiting
c40e8a94f228: Waiting
69d13bb66a4e: Waiting
05cfa4dfe4e0: Waiting
f057ee848829: Waiting
38512380f3ae: Pulling fs layer
d78d453fce70: Waiting
38512380f3ae: Waiting
7375eb43508f: Pulling fs layer
7375eb43508f: Waiting
f9df1fafd224: Verifying Checksum
f9df1fafd224: Download complete
a645a4b887f9: Verifying Checksum
a645a4b887f9: Download complete
57db7fe0b522: Download complete
fe703b657a32: Verifying Checksum
fe703b657a32: Download complete
ad7df9457e38: Verifying Checksum
ad7df9457e38: Download complete
9b279b7a7520: Verifying Checksum
9b279b7a7520: Download complete
a22d25e0a88f: Verifying Checksum
a22d25e0a88f: Download complete
95283fb14fdd: Verifying Checksum
95283fb14fdd: Download complete
54401ce2cd58: Verifying Checksum
54401ce2cd58: Download complete
a9bb3fcb22c8: Verifying Checksum
a9bb3fcb22c8: Download complete
a415f739b308: Verifying Checksum
a415f739b308: Download complete
c7fa0aeea839: Verifying Checksum
c7fa0aeea839: Download complete
fe703b657a32: Pull complete
5add3b44d39b: Verifying Checksum
5add3b44d39b: Download complete
f9df1fafd224: Pull complete
a645a4b887f9: Pull complete
57db7fe0b522: Pull complete
b070e225a67d: Verifying Checksum
b070e225a67d: Download complete
15773f9a9d11: Verifying Checksum
15773f9a9d11: Download complete
c63732422ad3: Verifying Checksum
c63732422ad3: Download complete
a63805ca0e65: Download complete
086a99966bf6: Download complete
ba0755234ac7: Verifying Checksum
ba0755234ac7: Download complete
c40e8a94f228: Verifying Checksum
c40e8a94f228: Download complete
05cfa4dfe4e0: Verifying Checksum
05cfa4dfe4e0: Download complete
ad7df9457e38: Pull complete
1e96c343b06a: Verifying Checksum
1e96c343b06a: Download complete
f057ee848829: Verifying Checksum
f057ee848829: Download complete
c2ed4d189f98: Verifying Checksum
c2ed4d189f98: Download complete
d78d453fce70: Verifying Checksum
d78d453fce70: Download complete
38512380f3ae: Verifying Checksum
38512380f3ae: Download complete
7375eb43508f: Verifying Checksum
7375eb43508f: Download complete
69d13bb66a4e: Verifying Checksum
69d13bb66a4e: Download complete
5add3b44d39b: Pull complete
9b279b7a7520: Pull complete
a22d25e0a88f: Pull complete
54401ce2cd58: Pull complete
95283fb14fdd: Pull complete
a415f739b308: Pull complete
a9bb3fcb22c8: Pull complete
c63732422ad3: Pull complete
c7fa0aeea839: Pull complete
15773f9a9d11: Pull complete
b070e225a67d: Pull complete
a63805ca0e65: Pull complete
086a99966bf6: Pull complete
ba0755234ac7: Pull complete
c2ed4d189f98: Pull complete
c40e8a94f228: Pull complete
69d13bb66a4e: Pull complete
05cfa4dfe4e0: Pull complete
1e96c343b06a: Pull complete
f057ee848829: Pull complete
d78d453fce70: Pull complete
38512380f3ae: Pull complete
7375eb43508f: Pull complete
Digest: sha256:b3b47805231a305d0f40c4bf069df20f6a2635574e6d4259fac651d3f9f6e098
Status: Downloaded newer image for googleapis/artman:latest
2020-05-10 05:12:09,183 synthtool > Cloning googleapis.
2020-05-10 05:12:10,808 synthtool > Running generator for google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-05-10 05:12:39,586 synthtool > Generated code into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2beta1.
2020-05-10 05:12:39,587 synthtool > Placed proto files into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2beta1/google/cloud/dialogflow_v2beta1/proto.
2020-05-10 05:12:40,053 synthtool > No replacements made in ['docs/**/*.rst', 'docs/conf.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-10 05:12:40,055 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2beta1/api.rst.
2020-05-10 05:12:40,055 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2beta1/types.rst.
2020-05-10 05:12:40,066 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2beta1/**/*.py'] for pattern import google.cloud.dialogflow, maybe replacement is no longer needed?
2020-05-10 05:12:40,068 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_session_entity_types_client_v2beta1.py.
2020-05-10 05:12:40,068 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_agents_client_v2beta1.py.
2020-05-10 05:12:40,068 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_environments_client_v2beta1.py.
2020-05-10 05:12:40,069 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_entity_types_client_v2beta1.py.
2020-05-10 05:12:40,069 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_intents_client_v2beta1.py.
2020-05-10 05:12:40,069 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_knowledge_bases_client_v2beta1.py.
2020-05-10 05:12:40,070 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_sessions_client_v2beta1.py.
2020-05-10 05:12:40,070 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_contexts_client_v2beta1.py.
2020-05-10 05:12:40,070 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_documents_client_v2beta1.py.
2020-05-10 05:12:40,071 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/types.py.
2020-05-10 05:12:40,071 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/__init__.py.
2020-05-10 05:12:40,071 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_entity_type_pb2_grpc.py.
2020-05-10 05:12:40,072 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_entity_type_pb2.py.
2020-05-10 05:12:40,072 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/environment_pb2.py.
2020-05-10 05:12:40,073 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/intent_pb2.py.
2020-05-10 05:12:40,074 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/intent_pb2_grpc.py.
2020-05-10 05:12:40,074 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/webhook_pb2.py.
2020-05-10 05:12:40,074 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/entity_type_pb2_grpc.py.
2020-05-10 05:12:40,075 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/document_pb2.py.
2020-05-10 05:12:40,075 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/context_pb2_grpc.py.
2020-05-10 05:12:40,076 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/agent_pb2_grpc.py.
2020-05-10 05:12:40,076 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/document_pb2_grpc.py.
2020-05-10 05:12:40,076 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/agent_pb2.py.
2020-05-10 05:12:40,077 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_pb2.py.
2020-05-10 05:12:40,077 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/knowledge_base_pb2_grpc.py.
2020-05-10 05:12:40,078 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_pb2_grpc.py.
2020-05-10 05:12:40,078 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/environment_pb2_grpc.py.
2020-05-10 05:12:40,079 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-10 05:12:40,079 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-10 05:12:40,079 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-10 05:12:40,080 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-10 05:12:40,080 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-10 05:12:40,081 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-10 05:12:40,082 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-10 05:12:40,082 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-10 05:12:40,082 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-10 05:12:40,083 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/knowledge_bases_grpc_transport.py.
2020-05-10 05:12:40,083 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/sessions_grpc_transport.py.
2020-05-10 05:12:40,083 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/environments_grpc_transport.py.
2020-05-10 05:12:40,084 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/session_entity_types_grpc_transport.py.
2020-05-10 05:12:40,084 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/contexts_grpc_transport.py.
2020-05-10 05:12:40,084 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/intents_grpc_transport.py.
2020-05-10 05:12:40,085 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/entity_types_grpc_transport.py.
2020-05-10 05:12:40,085 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/agents_grpc_transport.py.
2020-05-10 05:12:40,085 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/documents_grpc_transport.py.
2020-05-10 05:12:40,087 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_session_entity_types_client_v2beta1.py.
2020-05-10 05:12:40,087 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_agents_client_v2beta1.py.
2020-05-10 05:12:40,088 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_environments_client_v2beta1.py.
2020-05-10 05:12:40,088 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_entity_types_client_v2beta1.py.
2020-05-10 05:12:40,088 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_intents_client_v2beta1.py.
2020-05-10 05:12:40,088 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_knowledge_bases_client_v2beta1.py.
2020-05-10 05:12:40,089 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_sessions_client_v2beta1.py.
2020-05-10 05:12:40,089 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_contexts_client_v2beta1.py.
2020-05-10 05:12:40,089 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_documents_client_v2beta1.py.
2020-05-10 05:12:40,094 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-10 05:12:40,094 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-10 05:12:40,095 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-10 05:12:40,095 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-10 05:12:40,095 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-10 05:12:40,096 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-10 05:12:40,097 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-10 05:12:40,097 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-10 05:12:40,098 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-10 05:12:40,109 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2beta1/**/*.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-10 05:12:40,116 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-10 05:12:40,116 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-10 05:12:40,116 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-10 05:12:40,117 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-10 05:12:40,117 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-10 05:12:40,118 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-10 05:12:40,119 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-10 05:12:40,119 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-10 05:12:40,119 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-10 05:12:40,128 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-10 05:12:40,128 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-10 05:12:40,129 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-10 05:12:40,129 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-10 05:12:40,129 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-10 05:12:40,130 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-10 05:12:40,131 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-10 05:12:40,131 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-10 05:12:40,132 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-10 05:12:40,137 synthtool > No replacements made in dialogflow_v2beta1/gapic/agents_client.py for pattern (\s+).. raw:: html\s+<pre>curl, maybe replacement is no longer needed?
2020-05-10 05:12:40,137 synthtool > Running generator for google/cloud/dialogflow/v2/artman_dialogflow_v2.yaml.
2020-05-10 05:13:03,822 synthtool > Generated code into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2.
2020-05-10 05:13:03,822 synthtool > Placed proto files into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2/google/cloud/dialogflow_v2/proto.
2020-05-10 05:13:04,267 synthtool > No replacements made in ['docs/**/*.rst', 'docs/conf.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-10 05:13:04,267 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2/api.rst.
2020-05-10 05:13:04,268 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2/types.rst.
2020-05-10 05:13:04,277 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2/**/*.py'] for pattern import google.cloud.dialogflow, maybe replacement is no longer needed?
2020-05-10 05:13:04,277 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_sessions_client_v2.py.
2020-05-10 05:13:04,278 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_entity_types_client_v2.py.
2020-05-10 05:13:04,278 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_contexts_client_v2.py.
2020-05-10 05:13:04,278 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_session_entity_types_client_v2.py.
2020-05-10 05:13:04,279 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_intents_client_v2.py.
2020-05-10 05:13:04,279 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_agents_client_v2.py.
2020-05-10 05:13:04,279 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_environments_client_v2.py.
2020-05-10 05:13:04,280 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/types.py.
2020-05-10 05:13:04,281 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/__init__.py.
2020-05-10 05:13:04,281 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_entity_type_pb2_grpc.py.
2020-05-10 05:13:04,281 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_entity_type_pb2.py.
2020-05-10 05:13:04,282 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/intent_pb2.py.
2020-05-10 05:13:04,283 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/intent_pb2_grpc.py.
2020-05-10 05:13:04,283 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/webhook_pb2.py.
2020-05-10 05:13:04,283 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/entity_type_pb2_grpc.py.
2020-05-10 05:13:04,283 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/context_pb2_grpc.py.
2020-05-10 05:13:04,284 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/agent_pb2_grpc.py.
2020-05-10 05:13:04,284 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/agent_pb2.py.
2020-05-10 05:13:04,285 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_pb2.py.
2020-05-10 05:13:04,285 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_pb2_grpc.py.
2020-05-10 05:13:04,286 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/environment_pb2_grpc.py.
2020-05-10 05:13:04,286 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/agents_client.py.
2020-05-10 05:13:04,286 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/intents_client.py.
2020-05-10 05:13:04,287 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-10 05:13:04,287 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-10 05:13:04,289 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/environments_client.py.
2020-05-10 05:13:04,289 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/sessions_client.py.
2020-05-10 05:13:04,290 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/contexts_client.py.
2020-05-10 05:13:04,290 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/sessions_grpc_transport.py.
2020-05-10 05:13:04,290 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/environments_grpc_transport.py.
2020-05-10 05:13:04,291 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/session_entity_types_grpc_transport.py.
2020-05-10 05:13:04,291 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/contexts_grpc_transport.py.
2020-05-10 05:13:04,291 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/intents_grpc_transport.py.
2020-05-10 05:13:04,291 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/entity_types_grpc_transport.py.
2020-05-10 05:13:04,291 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/agents_grpc_transport.py.
2020-05-10 05:13:04,292 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_sessions_client_v2.py.
2020-05-10 05:13:04,292 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_entity_types_client_v2.py.
2020-05-10 05:13:04,293 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_contexts_client_v2.py.
2020-05-10 05:13:04,293 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_session_entity_types_client_v2.py.
2020-05-10 05:13:04,293 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_intents_client_v2.py.
2020-05-10 05:13:04,293 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_agents_client_v2.py.
2020-05-10 05:13:04,294 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_environments_client_v2.py.
2020-05-10 05:13:04,298 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/agents_client.py.
2020-05-10 05:13:04,299 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/intents_client.py.
2020-05-10 05:13:04,299 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-10 05:13:04,299 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-10 05:13:04,300 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/environments_client.py.
2020-05-10 05:13:04,300 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/sessions_client.py.
2020-05-10 05:13:04,301 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/contexts_client.py.
2020-05-10 05:13:04,307 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/agents_client.py.
2020-05-10 05:13:04,308 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/intents_client.py.
2020-05-10 05:13:04,308 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-10 05:13:04,308 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-10 05:13:04,309 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/environments_client.py.
2020-05-10 05:13:04,309 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/sessions_client.py.
2020-05-10 05:13:04,310 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/contexts_client.py.
2020-05-10 05:13:04,319 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2/**/*.py'] for pattern '-dialogflow', maybe replacement is no longer needed?
2020-05-10 05:13:04,325 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/agents_client.py.
2020-05-10 05:13:04,325 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/intents_client.py.
2020-05-10 05:13:04,326 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-10 05:13:04,326 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-10 05:13:04,327 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/environments_client.py.
2020-05-10 05:13:04,327 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/sessions_client.py.
2020-05-10 05:13:04,328 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/contexts_client.py.
2020-05-10 05:13:04,332 synthtool > No replacements made in dialogflow_v2/gapic/agents_client.py for pattern (\s+).. raw:: html\s+<pre>curl, maybe replacement is no longer needed?
2020-05-10 05:13:04,333 synthtool > Replaced '# Generated by the .*' in dialogflow_v2beta1/proto/session_pb2.py.
2020-05-10 05:13:04,333 synthtool > Replaced '# Generated by the .*' in dialogflow_v2beta1/proto/intent_pb2_grpc.py.
2020-05-10 05:13:04,334 synthtool > Replaced '# Generated by the .*' in dialogflow_v2/proto/intent_pb2_grpc.py.
2020-05-10 05:13:04,334 synthtool > Replaced '# Generated by the .*' in dialogflow_v2/proto/session_pb2.py.
2020-05-10 05:13:04,335 synthtool > No replacements made in ['dialogflow_v2beta1/gapic/intents_client.py', 'dialogflow_v2beta1/gapic/sessions_client.py', 'dialogflow_v2/gapic/intents_client.py'] for pattern # Copyright 2018 Google LLC, maybe replacement is no longer needed?
2020-05-10 05:13:04,336 synthtool > No replacements made in dialogflow_v2/gapic/agents_client.py for pattern }\\\" [\\]\n(\s+retry \(Optional), maybe replacement is no longer needed?
2020-05-10 05:13:04,342 synthtool > No replacements made in dialogflow_v2beta1/proto/intent_pb2.py for pattern \s+-----------, maybe replacement is no longer needed?
2020-05-10 05:13:04,343 synthtool > No replacements made in dialogflow_*/proto/session_pb2.py for pattern ============================================================================, maybe replacement is no longer needed?
2020-05-10 05:13:04,345 synthtool > Replaced '\\s*\\<\\>`__' in dialogflow_v2beta1/proto/audio_config_pb2.py.
2020-05-10 05:13:04,346 synthtool > Replaced '\\s*\\<\\>`__' in dialogflow_v2/proto/audio_config_pb2.py.
2020-05-10 05:13:04,347 synthtool > No replacements made in dialogflow_v2/proto/agent_pb2.py for pattern :math:, maybe replacement is no longer needed?
2020-05-10 05:13:04,347 synthtool > No replacements made in dialogflow_v2/proto/agent_pb2.py for pattern :raw-latex:, maybe replacement is no longer needed?
Error while collecting sessions.
Sessions not found: blacken
2020-05-10 05:13:04,554 synthtool > Failed executing nox -s blacken:

None
2020-05-10 05:13:04,555 synthtool > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py", line 122, in <module>
    s.shell.run(["nox", "-s", "blacken"], hide_output=False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['nox', '-s', 'blacken']' returned non-zero exit status 3.
2020-05-10 05:13:04 [ERROR] Synthesis failed
2020-05-10 05:13:04,590 autosynth > Synthesis failed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 599, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 471, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 549, in _inner_main
    ).synthesize(base_synth_log_path)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 118, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](https://sponge/ba0b5c63-ccbb-4373-a2fa-5105a99e88f0).

Autosynth is still having trouble generating dialogflow-python-client-v2. :sob:

Here's the output from running `synth.py`:

```
2020-05-11 05:11:34 [INFO] logs will be written to: /tmpfs/src/github/synthtool/logs/googleapis/dialogflow-python-client-v2
2020-05-11 05:11:34,407 autosynth > logs will be written to: /tmpfs/src/github/synthtool/logs/googleapis/dialogflow-python-client-v2
Switched to branch 'autosynth'
2020-05-11 05:11:35 [INFO] Running synthtool
2020-05-11 05:11:35,495 autosynth > Running synthtool
2020-05-11 05:11:35 [INFO] ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-11 05:11:35,496 autosynth > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-11 05:11:35,721 synthtool > Executing /home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py.
On branch autosynth
nothing to commit, working tree clean
2020-05-11 05:11:35,872 synthtool > Ensuring dependencies.
2020-05-11 05:11:35,876 synthtool > Pulling artman image.
latest: Pulling from googleapis/artman
fe703b657a32: Pulling fs layer
f9df1fafd224: Pulling fs layer
a645a4b887f9: Pulling fs layer
57db7fe0b522: Pulling fs layer
ad7df9457e38: Pulling fs layer
5add3b44d39b: Pulling fs layer
9b279b7a7520: Pulling fs layer
a22d25e0a88f: Pulling fs layer
54401ce2cd58: Pulling fs layer
95283fb14fdd: Pulling fs layer
a415f739b308: Pulling fs layer
a9bb3fcb22c8: Pulling fs layer
c63732422ad3: Pulling fs layer
c7fa0aeea839: Pulling fs layer
15773f9a9d11: Pulling fs layer
b070e225a67d: Pulling fs layer
a63805ca0e65: Pulling fs layer
086a99966bf6: Pulling fs layer
ba0755234ac7: Pulling fs layer
c2ed4d189f98: Pulling fs layer
c40e8a94f228: Pulling fs layer
69d13bb66a4e: Pulling fs layer
05cfa4dfe4e0: Pulling fs layer
1e96c343b06a: Pulling fs layer
f057ee848829: Pulling fs layer
d78d453fce70: Pulling fs layer
38512380f3ae: Pulling fs layer
7375eb43508f: Pulling fs layer
57db7fe0b522: Waiting
ad7df9457e38: Waiting
b070e225a67d: Waiting
a63805ca0e65: Waiting
086a99966bf6: Waiting
ba0755234ac7: Waiting
5add3b44d39b: Waiting
c2ed4d189f98: Waiting
c40e8a94f228: Waiting
69d13bb66a4e: Waiting
05cfa4dfe4e0: Waiting
9b279b7a7520: Waiting
1e96c343b06a: Waiting
a22d25e0a88f: Waiting
7375eb43508f: Waiting
54401ce2cd58: Waiting
d78d453fce70: Waiting
c7fa0aeea839: Waiting
a415f739b308: Waiting
15773f9a9d11: Waiting
c63732422ad3: Waiting
a9bb3fcb22c8: Waiting
38512380f3ae: Waiting
f9df1fafd224: Verifying Checksum
f9df1fafd224: Download complete
a645a4b887f9: Verifying Checksum
a645a4b887f9: Download complete
57db7fe0b522: Verifying Checksum
57db7fe0b522: Download complete
fe703b657a32: Verifying Checksum
fe703b657a32: Download complete
ad7df9457e38: Verifying Checksum
ad7df9457e38: Download complete
9b279b7a7520: Verifying Checksum
9b279b7a7520: Download complete
a22d25e0a88f: Verifying Checksum
a22d25e0a88f: Download complete
95283fb14fdd: Verifying Checksum
95283fb14fdd: Download complete
54401ce2cd58: Verifying Checksum
54401ce2cd58: Download complete
a9bb3fcb22c8: Verifying Checksum
a9bb3fcb22c8: Download complete
a415f739b308: Verifying Checksum
a415f739b308: Download complete
c7fa0aeea839: Verifying Checksum
c7fa0aeea839: Download complete
fe703b657a32: Pull complete
5add3b44d39b: Verifying Checksum
5add3b44d39b: Download complete
f9df1fafd224: Pull complete
a645a4b887f9: Pull complete
57db7fe0b522: Pull complete
15773f9a9d11: Verifying Checksum
15773f9a9d11: Download complete
b070e225a67d: Verifying Checksum
b070e225a67d: Download complete
a63805ca0e65: Verifying Checksum
a63805ca0e65: Download complete
086a99966bf6: Verifying Checksum
086a99966bf6: Download complete
c63732422ad3: Verifying Checksum
c63732422ad3: Download complete
ba0755234ac7: Verifying Checksum
ba0755234ac7: Download complete
c40e8a94f228: Verifying Checksum
c40e8a94f228: Download complete
05cfa4dfe4e0: Verifying Checksum
05cfa4dfe4e0: Download complete
ad7df9457e38: Pull complete
1e96c343b06a: Verifying Checksum
1e96c343b06a: Download complete
f057ee848829: Verifying Checksum
f057ee848829: Download complete
c2ed4d189f98: Verifying Checksum
c2ed4d189f98: Download complete
d78d453fce70: Verifying Checksum
d78d453fce70: Download complete
38512380f3ae: Verifying Checksum
38512380f3ae: Download complete
7375eb43508f: Verifying Checksum
7375eb43508f: Download complete
69d13bb66a4e: Verifying Checksum
69d13bb66a4e: Download complete
5add3b44d39b: Pull complete
9b279b7a7520: Pull complete
a22d25e0a88f: Pull complete
54401ce2cd58: Pull complete
95283fb14fdd: Pull complete
a415f739b308: Pull complete
a9bb3fcb22c8: Pull complete
c63732422ad3: Pull complete
c7fa0aeea839: Pull complete
15773f9a9d11: Pull complete
b070e225a67d: Pull complete
a63805ca0e65: Pull complete
086a99966bf6: Pull complete
ba0755234ac7: Pull complete
c2ed4d189f98: Pull complete
c40e8a94f228: Pull complete
69d13bb66a4e: Pull complete
05cfa4dfe4e0: Pull complete
1e96c343b06a: Pull complete
f057ee848829: Pull complete
d78d453fce70: Pull complete
38512380f3ae: Pull complete
7375eb43508f: Pull complete
Digest: sha256:b3b47805231a305d0f40c4bf069df20f6a2635574e6d4259fac651d3f9f6e098
Status: Downloaded newer image for googleapis/artman:latest
2020-05-11 05:12:13,562 synthtool > Cloning googleapis.
2020-05-11 05:12:15,702 synthtool > Running generator for google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-05-11 05:12:44,757 synthtool > Generated code into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2beta1.
2020-05-11 05:12:44,758 synthtool > Placed proto files into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2beta1/google/cloud/dialogflow_v2beta1/proto.
2020-05-11 05:12:45,242 synthtool > No replacements made in ['docs/**/*.rst', 'docs/conf.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-11 05:12:45,243 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2beta1/api.rst.
2020-05-11 05:12:45,244 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2beta1/types.rst.
2020-05-11 05:12:45,255 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2beta1/**/*.py'] for pattern import google.cloud.dialogflow, maybe replacement is no longer needed?
2020-05-11 05:12:45,257 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_session_entity_types_client_v2beta1.py.
2020-05-11 05:12:45,257 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_agents_client_v2beta1.py.
2020-05-11 05:12:45,257 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_environments_client_v2beta1.py.
2020-05-11 05:12:45,258 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_entity_types_client_v2beta1.py.
2020-05-11 05:12:45,258 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_intents_client_v2beta1.py.
2020-05-11 05:12:45,258 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_knowledge_bases_client_v2beta1.py.
2020-05-11 05:12:45,258 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_sessions_client_v2beta1.py.
2020-05-11 05:12:45,259 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_contexts_client_v2beta1.py.
2020-05-11 05:12:45,259 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_documents_client_v2beta1.py.
2020-05-11 05:12:45,259 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/types.py.
2020-05-11 05:12:45,259 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/__init__.py.
2020-05-11 05:12:45,260 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_entity_type_pb2_grpc.py.
2020-05-11 05:12:45,260 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_entity_type_pb2.py.
2020-05-11 05:12:45,260 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/environment_pb2.py.
2020-05-11 05:12:45,261 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/intent_pb2.py.
2020-05-11 05:12:45,262 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/intent_pb2_grpc.py.
2020-05-11 05:12:45,262 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/webhook_pb2.py.
2020-05-11 05:12:45,263 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/entity_type_pb2_grpc.py.
2020-05-11 05:12:45,263 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/document_pb2.py.
2020-05-11 05:12:45,263 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/context_pb2_grpc.py.
2020-05-11 05:12:45,264 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/agent_pb2_grpc.py.
2020-05-11 05:12:45,264 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/document_pb2_grpc.py.
2020-05-11 05:12:45,265 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/agent_pb2.py.
2020-05-11 05:12:45,265 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_pb2.py.
2020-05-11 05:12:45,266 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/knowledge_base_pb2_grpc.py.
2020-05-11 05:12:45,266 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_pb2_grpc.py.
2020-05-11 05:12:45,266 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/environment_pb2_grpc.py.
2020-05-11 05:12:45,267 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-11 05:12:45,267 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-11 05:12:45,267 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-11 05:12:45,268 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-11 05:12:45,268 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-11 05:12:45,269 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-11 05:12:45,269 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-11 05:12:45,270 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-11 05:12:45,270 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-11 05:12:45,271 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/knowledge_bases_grpc_transport.py.
2020-05-11 05:12:45,271 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/sessions_grpc_transport.py.
2020-05-11 05:12:45,271 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/environments_grpc_transport.py.
2020-05-11 05:12:45,271 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/session_entity_types_grpc_transport.py.
2020-05-11 05:12:45,272 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/contexts_grpc_transport.py.
2020-05-11 05:12:45,272 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/intents_grpc_transport.py.
2020-05-11 05:12:45,272 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/entity_types_grpc_transport.py.
2020-05-11 05:12:45,272 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/agents_grpc_transport.py.
2020-05-11 05:12:45,273 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/documents_grpc_transport.py.
2020-05-11 05:12:45,274 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_session_entity_types_client_v2beta1.py.
2020-05-11 05:12:45,274 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_agents_client_v2beta1.py.
2020-05-11 05:12:45,275 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_environments_client_v2beta1.py.
2020-05-11 05:12:45,275 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_entity_types_client_v2beta1.py.
2020-05-11 05:12:45,275 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_intents_client_v2beta1.py.
2020-05-11 05:12:45,275 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_knowledge_bases_client_v2beta1.py.
2020-05-11 05:12:45,276 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_sessions_client_v2beta1.py.
2020-05-11 05:12:45,276 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_contexts_client_v2beta1.py.
2020-05-11 05:12:45,276 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_documents_client_v2beta1.py.
2020-05-11 05:12:45,281 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-11 05:12:45,281 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-11 05:12:45,281 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-11 05:12:45,282 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-11 05:12:45,282 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-11 05:12:45,283 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-11 05:12:45,283 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-11 05:12:45,284 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-11 05:12:45,284 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-11 05:12:45,295 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2beta1/**/*.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-11 05:12:45,301 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-11 05:12:45,302 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-11 05:12:45,302 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-11 05:12:45,302 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-11 05:12:45,303 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-11 05:12:45,303 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-11 05:12:45,304 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-11 05:12:45,304 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-11 05:12:45,305 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-11 05:12:45,313 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-11 05:12:45,313 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-11 05:12:45,314 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-11 05:12:45,314 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-11 05:12:45,314 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-11 05:12:45,315 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-11 05:12:45,316 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-11 05:12:45,316 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-11 05:12:45,316 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-11 05:12:45,322 synthtool > No replacements made in dialogflow_v2beta1/gapic/agents_client.py for pattern (\s+).. raw:: html\s+<pre>curl, maybe replacement is no longer needed?
2020-05-11 05:12:45,322 synthtool > Running generator for google/cloud/dialogflow/v2/artman_dialogflow_v2.yaml.
2020-05-11 05:13:08,392 synthtool > Generated code into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2.
2020-05-11 05:13:08,392 synthtool > Placed proto files into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2/google/cloud/dialogflow_v2/proto.
2020-05-11 05:13:08,847 synthtool > No replacements made in ['docs/**/*.rst', 'docs/conf.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-11 05:13:08,848 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2/api.rst.
2020-05-11 05:13:08,849 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2/types.rst.
2020-05-11 05:13:08,858 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2/**/*.py'] for pattern import google.cloud.dialogflow, maybe replacement is no longer needed?
2020-05-11 05:13:08,858 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_sessions_client_v2.py.
2020-05-11 05:13:08,858 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_entity_types_client_v2.py.
2020-05-11 05:13:08,859 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_contexts_client_v2.py.
2020-05-11 05:13:08,859 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_session_entity_types_client_v2.py.
2020-05-11 05:13:08,859 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_intents_client_v2.py.
2020-05-11 05:13:08,859 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_agents_client_v2.py.
2020-05-11 05:13:08,860 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_environments_client_v2.py.
2020-05-11 05:13:08,861 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/types.py.
2020-05-11 05:13:08,861 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/__init__.py.
2020-05-11 05:13:08,862 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_entity_type_pb2_grpc.py.
2020-05-11 05:13:08,862 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_entity_type_pb2.py.
2020-05-11 05:13:08,863 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/intent_pb2.py.
2020-05-11 05:13:08,863 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/intent_pb2_grpc.py.
2020-05-11 05:13:08,864 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/webhook_pb2.py.
2020-05-11 05:13:08,864 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/entity_type_pb2_grpc.py.
2020-05-11 05:13:08,864 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/context_pb2_grpc.py.
2020-05-11 05:13:08,865 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/agent_pb2_grpc.py.
2020-05-11 05:13:08,865 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/agent_pb2.py.
2020-05-11 05:13:08,866 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_pb2.py.
2020-05-11 05:13:08,866 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_pb2_grpc.py.
2020-05-11 05:13:08,866 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/environment_pb2_grpc.py.
2020-05-11 05:13:08,867 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/agents_client.py.
2020-05-11 05:13:08,867 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/intents_client.py.
2020-05-11 05:13:08,867 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-11 05:13:08,868 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-11 05:13:08,870 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/environments_client.py.
2020-05-11 05:13:08,870 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/sessions_client.py.
2020-05-11 05:13:08,870 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/contexts_client.py.
2020-05-11 05:13:08,871 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/sessions_grpc_transport.py.
2020-05-11 05:13:08,871 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/environments_grpc_transport.py.
2020-05-11 05:13:08,871 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/session_entity_types_grpc_transport.py.
2020-05-11 05:13:08,872 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/contexts_grpc_transport.py.
2020-05-11 05:13:08,872 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/intents_grpc_transport.py.
2020-05-11 05:13:08,872 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/entity_types_grpc_transport.py.
2020-05-11 05:13:08,872 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/agents_grpc_transport.py.
2020-05-11 05:13:08,873 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_sessions_client_v2.py.
2020-05-11 05:13:08,873 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_entity_types_client_v2.py.
2020-05-11 05:13:08,873 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_contexts_client_v2.py.
2020-05-11 05:13:08,874 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_session_entity_types_client_v2.py.
2020-05-11 05:13:08,874 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_intents_client_v2.py.
2020-05-11 05:13:08,874 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_agents_client_v2.py.
2020-05-11 05:13:08,875 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_environments_client_v2.py.
2020-05-11 05:13:08,879 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/agents_client.py.
2020-05-11 05:13:08,879 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/intents_client.py.
2020-05-11 05:13:08,880 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-11 05:13:08,880 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-11 05:13:08,881 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/environments_client.py.
2020-05-11 05:13:08,881 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/sessions_client.py.
2020-05-11 05:13:08,882 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/contexts_client.py.
2020-05-11 05:13:08,888 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/agents_client.py.
2020-05-11 05:13:08,889 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/intents_client.py.
2020-05-11 05:13:08,889 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-11 05:13:08,889 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-11 05:13:08,890 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/environments_client.py.
2020-05-11 05:13:08,890 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/sessions_client.py.
2020-05-11 05:13:08,891 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/contexts_client.py.
2020-05-11 05:13:08,900 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2/**/*.py'] for pattern '-dialogflow', maybe replacement is no longer needed?
2020-05-11 05:13:08,907 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/agents_client.py.
2020-05-11 05:13:08,907 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/intents_client.py.
2020-05-11 05:13:08,907 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-11 05:13:08,908 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-11 05:13:08,909 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/environments_client.py.
2020-05-11 05:13:08,909 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/sessions_client.py.
2020-05-11 05:13:08,910 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/contexts_client.py.
2020-05-11 05:13:08,915 synthtool > No replacements made in dialogflow_v2/gapic/agents_client.py for pattern (\s+).. raw:: html\s+<pre>curl, maybe replacement is no longer needed?
2020-05-11 05:13:08,916 synthtool > Replaced '# Generated by the .*' in dialogflow_v2beta1/proto/session_pb2.py.
2020-05-11 05:13:08,916 synthtool > Replaced '# Generated by the .*' in dialogflow_v2beta1/proto/intent_pb2_grpc.py.
2020-05-11 05:13:08,916 synthtool > Replaced '# Generated by the .*' in dialogflow_v2/proto/intent_pb2_grpc.py.
2020-05-11 05:13:08,917 synthtool > Replaced '# Generated by the .*' in dialogflow_v2/proto/session_pb2.py.
2020-05-11 05:13:08,918 synthtool > No replacements made in ['dialogflow_v2beta1/gapic/intents_client.py', 'dialogflow_v2beta1/gapic/sessions_client.py', 'dialogflow_v2/gapic/intents_client.py'] for pattern # Copyright 2018 Google LLC, maybe replacement is no longer needed?
2020-05-11 05:13:08,919 synthtool > No replacements made in dialogflow_v2/gapic/agents_client.py for pattern }\\\" [\\]\n(\s+retry \(Optional), maybe replacement is no longer needed?
2020-05-11 05:13:08,925 synthtool > No replacements made in dialogflow_v2beta1/proto/intent_pb2.py for pattern \s+-----------, maybe replacement is no longer needed?
2020-05-11 05:13:08,926 synthtool > No replacements made in dialogflow_*/proto/session_pb2.py for pattern ============================================================================, maybe replacement is no longer needed?
2020-05-11 05:13:08,928 synthtool > Replaced '\\s*\\<\\>`__' in dialogflow_v2beta1/proto/audio_config_pb2.py.
2020-05-11 05:13:08,929 synthtool > Replaced '\\s*\\<\\>`__' in dialogflow_v2/proto/audio_config_pb2.py.
2020-05-11 05:13:08,930 synthtool > No replacements made in dialogflow_v2/proto/agent_pb2.py for pattern :math:, maybe replacement is no longer needed?
2020-05-11 05:13:08,930 synthtool > No replacements made in dialogflow_v2/proto/agent_pb2.py for pattern :raw-latex:, maybe replacement is no longer needed?
Error while collecting sessions.
Sessions not found: blacken
2020-05-11 05:13:09,141 synthtool > Failed executing nox -s blacken:

None
2020-05-11 05:13:09,143 synthtool > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py", line 122, in <module>
    s.shell.run(["nox", "-s", "blacken"], hide_output=False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['nox', '-s', 'blacken']' returned non-zero exit status 3.
2020-05-11 05:13:09 [ERROR] Synthesis failed
2020-05-11 05:13:09,177 autosynth > Synthesis failed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 599, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 471, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 549, in _inner_main
    ).synthesize(base_synth_log_path)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 118, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](https://sponge/9a4822b4-39a6-49c2-b189-6673a37c2643).

Autosynth is still having trouble generating dialogflow-python-client-v2. :sob:

Here's the output from running `synth.py`:

```
2020-05-12 05:11:33 [INFO] logs will be written to: /tmpfs/src/github/synthtool/logs/googleapis/dialogflow-python-client-v2
2020-05-12 05:11:33,937 autosynth > logs will be written to: /tmpfs/src/github/synthtool/logs/googleapis/dialogflow-python-client-v2
Switched to branch 'autosynth'
2020-05-12 05:11:34 [INFO] Running synthtool
2020-05-12 05:11:34,759 autosynth > Running synthtool
2020-05-12 05:11:34 [INFO] ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-12 05:11:34,759 autosynth > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-12 05:11:35,038 synthtool > Executing /home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py.
On branch autosynth
nothing to commit, working tree clean
2020-05-12 05:11:35,202 synthtool > Ensuring dependencies.
2020-05-12 05:11:35,206 synthtool > Pulling artman image.
latest: Pulling from googleapis/artman
fe703b657a32: Pulling fs layer
f9df1fafd224: Pulling fs layer
a645a4b887f9: Pulling fs layer
57db7fe0b522: Pulling fs layer
ad7df9457e38: Pulling fs layer
5add3b44d39b: Pulling fs layer
9b279b7a7520: Pulling fs layer
a22d25e0a88f: Pulling fs layer
54401ce2cd58: Pulling fs layer
95283fb14fdd: Pulling fs layer
a415f739b308: Pulling fs layer
a9bb3fcb22c8: Pulling fs layer
c63732422ad3: Pulling fs layer
c7fa0aeea839: Pulling fs layer
15773f9a9d11: Pulling fs layer
b070e225a67d: Pulling fs layer
a63805ca0e65: Pulling fs layer
086a99966bf6: Pulling fs layer
ba0755234ac7: Pulling fs layer
c2ed4d189f98: Pulling fs layer
57db7fe0b522: Waiting
c40e8a94f228: Pulling fs layer
69d13bb66a4e: Pulling fs layer
05cfa4dfe4e0: Pulling fs layer
1e96c343b06a: Pulling fs layer
f057ee848829: Pulling fs layer
d78d453fce70: Pulling fs layer
38512380f3ae: Pulling fs layer
7375eb43508f: Pulling fs layer
a9bb3fcb22c8: Waiting
c63732422ad3: Waiting
c7fa0aeea839: Waiting
15773f9a9d11: Waiting
b070e225a67d: Waiting
ad7df9457e38: Waiting
a63805ca0e65: Waiting
086a99966bf6: Waiting
5add3b44d39b: Waiting
9b279b7a7520: Waiting
ba0755234ac7: Waiting
c2ed4d189f98: Waiting
95283fb14fdd: Waiting
a22d25e0a88f: Waiting
c40e8a94f228: Waiting
54401ce2cd58: Waiting
69d13bb66a4e: Waiting
38512380f3ae: Waiting
f057ee848829: Waiting
05cfa4dfe4e0: Waiting
d78d453fce70: Waiting
1e96c343b06a: Waiting
7375eb43508f: Waiting
a645a4b887f9: Verifying Checksum
a645a4b887f9: Download complete
f9df1fafd224: Verifying Checksum
f9df1fafd224: Download complete
57db7fe0b522: Verifying Checksum
57db7fe0b522: Download complete
fe703b657a32: Verifying Checksum
fe703b657a32: Download complete
ad7df9457e38: Download complete
9b279b7a7520: Verifying Checksum
9b279b7a7520: Download complete
a22d25e0a88f: Verifying Checksum
a22d25e0a88f: Download complete
95283fb14fdd: Verifying Checksum
95283fb14fdd: Download complete
54401ce2cd58: Verifying Checksum
54401ce2cd58: Download complete
a9bb3fcb22c8: Verifying Checksum
a9bb3fcb22c8: Download complete
a415f739b308: Verifying Checksum
a415f739b308: Download complete
c7fa0aeea839: Verifying Checksum
c7fa0aeea839: Download complete
5add3b44d39b: Verifying Checksum
5add3b44d39b: Download complete
fe703b657a32: Pull complete
15773f9a9d11: Verifying Checksum
15773f9a9d11: Download complete
f9df1fafd224: Pull complete
a645a4b887f9: Pull complete
b070e225a67d: Verifying Checksum
b070e225a67d: Download complete
57db7fe0b522: Pull complete
a63805ca0e65: Verifying Checksum
a63805ca0e65: Download complete
c63732422ad3: Verifying Checksum
c63732422ad3: Download complete
086a99966bf6: Verifying Checksum
086a99966bf6: Download complete
ba0755234ac7: Verifying Checksum
ba0755234ac7: Download complete
c40e8a94f228: Download complete
05cfa4dfe4e0: Verifying Checksum
05cfa4dfe4e0: Download complete
1e96c343b06a: Verifying Checksum
1e96c343b06a: Download complete
ad7df9457e38: Pull complete
f057ee848829: Verifying Checksum
f057ee848829: Download complete
c2ed4d189f98: Verifying Checksum
c2ed4d189f98: Download complete
d78d453fce70: Verifying Checksum
d78d453fce70: Download complete
38512380f3ae: Verifying Checksum
38512380f3ae: Download complete
7375eb43508f: Verifying Checksum
7375eb43508f: Download complete
69d13bb66a4e: Verifying Checksum
69d13bb66a4e: Download complete
5add3b44d39b: Pull complete
9b279b7a7520: Pull complete
a22d25e0a88f: Pull complete
54401ce2cd58: Pull complete
95283fb14fdd: Pull complete
a415f739b308: Pull complete
a9bb3fcb22c8: Pull complete
c63732422ad3: Pull complete
c7fa0aeea839: Pull complete
15773f9a9d11: Pull complete
b070e225a67d: Pull complete
a63805ca0e65: Pull complete
086a99966bf6: Pull complete
ba0755234ac7: Pull complete
c2ed4d189f98: Pull complete
c40e8a94f228: Pull complete
69d13bb66a4e: Pull complete
05cfa4dfe4e0: Pull complete
1e96c343b06a: Pull complete
f057ee848829: Pull complete
d78d453fce70: Pull complete
38512380f3ae: Pull complete
7375eb43508f: Pull complete
Digest: sha256:b3b47805231a305d0f40c4bf069df20f6a2635574e6d4259fac651d3f9f6e098
Status: Downloaded newer image for googleapis/artman:latest
2020-05-12 05:12:16,434 synthtool > Cloning googleapis.
2020-05-12 05:12:18,110 synthtool > Running generator for google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-05-12 05:12:46,672 synthtool > Generated code into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2beta1.
2020-05-12 05:12:46,673 synthtool > Placed proto files into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2beta1/google/cloud/dialogflow_v2beta1/proto.
2020-05-12 05:12:47,152 synthtool > No replacements made in ['docs/**/*.rst', 'docs/conf.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-12 05:12:47,154 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2beta1/api.rst.
2020-05-12 05:12:47,154 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2beta1/types.rst.
2020-05-12 05:12:47,165 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2beta1/**/*.py'] for pattern import google.cloud.dialogflow, maybe replacement is no longer needed?
2020-05-12 05:12:47,167 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_session_entity_types_client_v2beta1.py.
2020-05-12 05:12:47,167 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_agents_client_v2beta1.py.
2020-05-12 05:12:47,167 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_environments_client_v2beta1.py.
2020-05-12 05:12:47,168 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_entity_types_client_v2beta1.py.
2020-05-12 05:12:47,168 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_intents_client_v2beta1.py.
2020-05-12 05:12:47,168 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_knowledge_bases_client_v2beta1.py.
2020-05-12 05:12:47,168 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_sessions_client_v2beta1.py.
2020-05-12 05:12:47,169 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_contexts_client_v2beta1.py.
2020-05-12 05:12:47,169 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_documents_client_v2beta1.py.
2020-05-12 05:12:47,169 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/types.py.
2020-05-12 05:12:47,170 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/__init__.py.
2020-05-12 05:12:47,170 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_entity_type_pb2_grpc.py.
2020-05-12 05:12:47,170 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_entity_type_pb2.py.
2020-05-12 05:12:47,171 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/environment_pb2.py.
2020-05-12 05:12:47,171 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/intent_pb2.py.
2020-05-12 05:12:47,172 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/intent_pb2_grpc.py.
2020-05-12 05:12:47,172 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/webhook_pb2.py.
2020-05-12 05:12:47,173 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/entity_type_pb2_grpc.py.
2020-05-12 05:12:47,173 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/document_pb2.py.
2020-05-12 05:12:47,173 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/context_pb2_grpc.py.
2020-05-12 05:12:47,174 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/agent_pb2_grpc.py.
2020-05-12 05:12:47,174 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/document_pb2_grpc.py.
2020-05-12 05:12:47,175 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/agent_pb2.py.
2020-05-12 05:12:47,175 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_pb2.py.
2020-05-12 05:12:47,176 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/knowledge_base_pb2_grpc.py.
2020-05-12 05:12:47,176 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_pb2_grpc.py.
2020-05-12 05:12:47,176 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/environment_pb2_grpc.py.
2020-05-12 05:12:47,177 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-12 05:12:47,177 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-12 05:12:47,178 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-12 05:12:47,178 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-12 05:12:47,178 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-12 05:12:47,179 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-12 05:12:47,180 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-12 05:12:47,180 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-12 05:12:47,181 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-12 05:12:47,181 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/knowledge_bases_grpc_transport.py.
2020-05-12 05:12:47,181 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/sessions_grpc_transport.py.
2020-05-12 05:12:47,182 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/environments_grpc_transport.py.
2020-05-12 05:12:47,182 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/session_entity_types_grpc_transport.py.
2020-05-12 05:12:47,182 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/contexts_grpc_transport.py.
2020-05-12 05:12:47,182 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/intents_grpc_transport.py.
2020-05-12 05:12:47,183 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/entity_types_grpc_transport.py.
2020-05-12 05:12:47,183 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/agents_grpc_transport.py.
2020-05-12 05:12:47,183 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/documents_grpc_transport.py.
2020-05-12 05:12:47,185 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_session_entity_types_client_v2beta1.py.
2020-05-12 05:12:47,185 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_agents_client_v2beta1.py.
2020-05-12 05:12:47,185 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_environments_client_v2beta1.py.
2020-05-12 05:12:47,186 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_entity_types_client_v2beta1.py.
2020-05-12 05:12:47,186 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_intents_client_v2beta1.py.
2020-05-12 05:12:47,186 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_knowledge_bases_client_v2beta1.py.
2020-05-12 05:12:47,186 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_sessions_client_v2beta1.py.
2020-05-12 05:12:47,187 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_contexts_client_v2beta1.py.
2020-05-12 05:12:47,187 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_documents_client_v2beta1.py.
2020-05-12 05:12:47,191 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-12 05:12:47,192 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-12 05:12:47,192 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-12 05:12:47,192 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-12 05:12:47,193 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-12 05:12:47,194 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-12 05:12:47,194 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-12 05:12:47,195 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-12 05:12:47,195 synthtool > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-12 05:12:47,207 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2beta1/**/*.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-12 05:12:47,213 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-12 05:12:47,214 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-12 05:12:47,214 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-12 05:12:47,214 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-12 05:12:47,215 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-12 05:12:47,215 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-12 05:12:47,216 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-12 05:12:47,216 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-12 05:12:47,217 synthtool > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-12 05:12:47,225 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-12 05:12:47,225 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-12 05:12:47,226 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-12 05:12:47,226 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-12 05:12:47,226 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-12 05:12:47,227 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-12 05:12:47,228 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-12 05:12:47,228 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-12 05:12:47,228 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-12 05:12:47,234 synthtool > No replacements made in dialogflow_v2beta1/gapic/agents_client.py for pattern (\s+).. raw:: html\s+<pre>curl, maybe replacement is no longer needed?
2020-05-12 05:12:47,234 synthtool > Running generator for google/cloud/dialogflow/v2/artman_dialogflow_v2.yaml.
2020-05-12 05:13:10,159 synthtool > Generated code into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2.
2020-05-12 05:13:10,160 synthtool > Placed proto files into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2/google/cloud/dialogflow_v2/proto.
2020-05-12 05:13:10,606 synthtool > No replacements made in ['docs/**/*.rst', 'docs/conf.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-12 05:13:10,607 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2/api.rst.
2020-05-12 05:13:10,607 synthtool > Replaced 'google.cloud.dialogflow' in docs/gapic/v2/types.rst.
2020-05-12 05:13:10,616 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2/**/*.py'] for pattern import google.cloud.dialogflow, maybe replacement is no longer needed?
2020-05-12 05:13:10,616 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_sessions_client_v2.py.
2020-05-12 05:13:10,616 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_entity_types_client_v2.py.
2020-05-12 05:13:10,617 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_contexts_client_v2.py.
2020-05-12 05:13:10,617 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_session_entity_types_client_v2.py.
2020-05-12 05:13:10,617 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_intents_client_v2.py.
2020-05-12 05:13:10,617 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_agents_client_v2.py.
2020-05-12 05:13:10,618 synthtool > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_environments_client_v2.py.
2020-05-12 05:13:10,619 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/types.py.
2020-05-12 05:13:10,619 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/__init__.py.
2020-05-12 05:13:10,620 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_entity_type_pb2_grpc.py.
2020-05-12 05:13:10,620 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_entity_type_pb2.py.
2020-05-12 05:13:10,621 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/intent_pb2.py.
2020-05-12 05:13:10,621 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/intent_pb2_grpc.py.
2020-05-12 05:13:10,621 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/webhook_pb2.py.
2020-05-12 05:13:10,622 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/entity_type_pb2_grpc.py.
2020-05-12 05:13:10,622 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/context_pb2_grpc.py.
2020-05-12 05:13:10,623 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/agent_pb2_grpc.py.
2020-05-12 05:13:10,623 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/agent_pb2.py.
2020-05-12 05:13:10,623 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_pb2.py.
2020-05-12 05:13:10,624 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_pb2_grpc.py.
2020-05-12 05:13:10,624 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/environment_pb2_grpc.py.
2020-05-12 05:13:10,624 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/agents_client.py.
2020-05-12 05:13:10,625 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/intents_client.py.
2020-05-12 05:13:10,625 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-12 05:13:10,625 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-12 05:13:10,627 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/environments_client.py.
2020-05-12 05:13:10,627 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/sessions_client.py.
2020-05-12 05:13:10,628 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/contexts_client.py.
2020-05-12 05:13:10,628 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/sessions_grpc_transport.py.
2020-05-12 05:13:10,628 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/environments_grpc_transport.py.
2020-05-12 05:13:10,629 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/session_entity_types_grpc_transport.py.
2020-05-12 05:13:10,629 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/contexts_grpc_transport.py.
2020-05-12 05:13:10,629 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/intents_grpc_transport.py.
2020-05-12 05:13:10,629 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/entity_types_grpc_transport.py.
2020-05-12 05:13:10,630 synthtool > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/agents_grpc_transport.py.
2020-05-12 05:13:10,630 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_sessions_client_v2.py.
2020-05-12 05:13:10,630 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_entity_types_client_v2.py.
2020-05-12 05:13:10,631 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_contexts_client_v2.py.
2020-05-12 05:13:10,631 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_session_entity_types_client_v2.py.
2020-05-12 05:13:10,631 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_intents_client_v2.py.
2020-05-12 05:13:10,631 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_agents_client_v2.py.
2020-05-12 05:13:10,632 synthtool > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_environments_client_v2.py.
2020-05-12 05:13:10,636 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/agents_client.py.
2020-05-12 05:13:10,637 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/intents_client.py.
2020-05-12 05:13:10,637 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-12 05:13:10,637 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-12 05:13:10,638 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/environments_client.py.
2020-05-12 05:13:10,639 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/sessions_client.py.
2020-05-12 05:13:10,639 synthtool > Replaced 'from google.cloud import' in dialogflow_v2/gapic/contexts_client.py.
2020-05-12 05:13:10,646 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/agents_client.py.
2020-05-12 05:13:10,646 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/intents_client.py.
2020-05-12 05:13:10,646 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-12 05:13:10,647 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-12 05:13:10,647 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/environments_client.py.
2020-05-12 05:13:10,648 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/sessions_client.py.
2020-05-12 05:13:10,648 synthtool > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/contexts_client.py.
2020-05-12 05:13:10,658 synthtool > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2/**/*.py'] for pattern '-dialogflow', maybe replacement is no longer needed?
2020-05-12 05:13:10,663 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/agents_client.py.
2020-05-12 05:13:10,664 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/intents_client.py.
2020-05-12 05:13:10,664 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-12 05:13:10,664 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-12 05:13:10,665 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/environments_client.py.
2020-05-12 05:13:10,665 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/sessions_client.py.
2020-05-12 05:13:10,666 synthtool > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/contexts_client.py.
2020-05-12 05:13:10,671 synthtool > No replacements made in dialogflow_v2/gapic/agents_client.py for pattern (\s+).. raw:: html\s+<pre>curl, maybe replacement is no longer needed?
2020-05-12 05:13:10,672 synthtool > Replaced '# Generated by the .*' in dialogflow_v2beta1/proto/session_pb2.py.
2020-05-12 05:13:10,672 synthtool > Replaced '# Generated by the .*' in dialogflow_v2beta1/proto/intent_pb2_grpc.py.
2020-05-12 05:13:10,672 synthtool > Replaced '# Generated by the .*' in dialogflow_v2/proto/intent_pb2_grpc.py.
2020-05-12 05:13:10,673 synthtool > Replaced '# Generated by the .*' in dialogflow_v2/proto/session_pb2.py.
2020-05-12 05:13:10,674 synthtool > No replacements made in ['dialogflow_v2beta1/gapic/intents_client.py', 'dialogflow_v2beta1/gapic/sessions_client.py', 'dialogflow_v2/gapic/intents_client.py'] for pattern # Copyright 2018 Google LLC, maybe replacement is no longer needed?
2020-05-12 05:13:10,674 synthtool > No replacements made in dialogflow_v2/gapic/agents_client.py for pattern }\\\" [\\]\n(\s+retry \(Optional), maybe replacement is no longer needed?
2020-05-12 05:13:10,681 synthtool > No replacements made in dialogflow_v2beta1/proto/intent_pb2.py for pattern \s+-----------, maybe replacement is no longer needed?
2020-05-12 05:13:10,682 synthtool > No replacements made in dialogflow_*/proto/session_pb2.py for pattern ============================================================================, maybe replacement is no longer needed?
2020-05-12 05:13:10,684 synthtool > Replaced '\\s*\\<\\>`__' in dialogflow_v2beta1/proto/audio_config_pb2.py.
2020-05-12 05:13:10,685 synthtool > Replaced '\\s*\\<\\>`__' in dialogflow_v2/proto/audio_config_pb2.py.
2020-05-12 05:13:10,686 synthtool > No replacements made in dialogflow_v2/proto/agent_pb2.py for pattern :math:, maybe replacement is no longer needed?
2020-05-12 05:13:10,686 synthtool > No replacements made in dialogflow_v2/proto/agent_pb2.py for pattern :raw-latex:, maybe replacement is no longer needed?
Error while collecting sessions.
Sessions not found: blacken
2020-05-12 05:13:10,898 synthtool > Failed executing nox -s blacken:

None
2020-05-12 05:13:10,899 synthtool > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py", line 122, in <module>
    s.shell.run(["nox", "-s", "blacken"], hide_output=False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['nox', '-s', 'blacken']' returned non-zero exit status 3.
2020-05-12 05:13:10 [ERROR] Synthesis failed
2020-05-12 05:13:10,931 autosynth > Synthesis failed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 599, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 471, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 549, in _inner_main
    ).synthesize(base_synth_log_path)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 118, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](https://sponge/6de24127-f414-470c-9ad4-eb75a97e1087).

Autosynth is still having trouble generating dialogflow-python-client-v2. :sob:

Here's the output from running `synth.py`:

```
2020-05-13 05:11:38,071 autosynth [INFO] > logs will be written to: /tmpfs/src/github/synthtool/logs/googleapis/dialogflow-python-client-v2
2020-05-13 05:11:38,688 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 05:11:38,691 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 05:11:38,694 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 05:11:38,697 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 05:11:38,700 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 05:11:38,703 autosynth [DEBUG] > Running: {' '.join(command)}
Switched to branch 'autosynth'
2020-05-13 05:11:38,907 autosynth [INFO] > Running synthtool
2020-05-13 05:11:38,907 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-13 05:11:38,911 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 05:11:39,117 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py.
On branch autosynth
nothing to commit, working tree clean
2020-05-13 05:11:39,278 synthtool [DEBUG] > Ensuring dependencies.
2020-05-13 05:11:39,283 synthtool [DEBUG] > Pulling artman image.
latest: Pulling from googleapis/artman
fe703b657a32: Pulling fs layer
f9df1fafd224: Pulling fs layer
a645a4b887f9: Pulling fs layer
57db7fe0b522: Pulling fs layer
ad7df9457e38: Pulling fs layer
5add3b44d39b: Pulling fs layer
9b279b7a7520: Pulling fs layer
a22d25e0a88f: Pulling fs layer
54401ce2cd58: Pulling fs layer
95283fb14fdd: Pulling fs layer
a415f739b308: Pulling fs layer
a9bb3fcb22c8: Pulling fs layer
c63732422ad3: Pulling fs layer
c7fa0aeea839: Pulling fs layer
15773f9a9d11: Pulling fs layer
b070e225a67d: Pulling fs layer
a63805ca0e65: Pulling fs layer
086a99966bf6: Pulling fs layer
ba0755234ac7: Pulling fs layer
c2ed4d189f98: Pulling fs layer
c40e8a94f228: Pulling fs layer
69d13bb66a4e: Pulling fs layer
05cfa4dfe4e0: Pulling fs layer
1e96c343b06a: Pulling fs layer
f057ee848829: Pulling fs layer
d78d453fce70: Pulling fs layer
38512380f3ae: Pulling fs layer
7375eb43508f: Pulling fs layer
c63732422ad3: Waiting
c7fa0aeea839: Waiting
a22d25e0a88f: Waiting
57db7fe0b522: Waiting
ad7df9457e38: Waiting
15773f9a9d11: Waiting
b070e225a67d: Waiting
5add3b44d39b: Waiting
9b279b7a7520: Waiting
54401ce2cd58: Waiting
a63805ca0e65: Waiting
95283fb14fdd: Waiting
086a99966bf6: Waiting
a415f739b308: Waiting
a9bb3fcb22c8: Waiting
ba0755234ac7: Waiting
f057ee848829: Waiting
7375eb43508f: Waiting
d78d453fce70: Waiting
c40e8a94f228: Waiting
38512380f3ae: Waiting
c2ed4d189f98: Waiting
05cfa4dfe4e0: Waiting
a645a4b887f9: Download complete
f9df1fafd224: Download complete
57db7fe0b522: Download complete
fe703b657a32: Verifying Checksum
fe703b657a32: Download complete
ad7df9457e38: Verifying Checksum
ad7df9457e38: Download complete
9b279b7a7520: Verifying Checksum
9b279b7a7520: Download complete
a22d25e0a88f: Verifying Checksum
a22d25e0a88f: Download complete
54401ce2cd58: Verifying Checksum
54401ce2cd58: Download complete
95283fb14fdd: Verifying Checksum
95283fb14fdd: Download complete
a9bb3fcb22c8: Verifying Checksum
a9bb3fcb22c8: Download complete
a415f739b308: Verifying Checksum
a415f739b308: Download complete
c7fa0aeea839: Verifying Checksum
c7fa0aeea839: Download complete
fe703b657a32: Pull complete
f9df1fafd224: Pull complete
5add3b44d39b: Verifying Checksum
5add3b44d39b: Download complete
a645a4b887f9: Pull complete
57db7fe0b522: Pull complete
b070e225a67d: Verifying Checksum
b070e225a67d: Download complete
15773f9a9d11: Verifying Checksum
15773f9a9d11: Download complete
a63805ca0e65: Download complete
086a99966bf6: Verifying Checksum
086a99966bf6: Download complete
ba0755234ac7: Verifying Checksum
ba0755234ac7: Download complete
ad7df9457e38: Pull complete
c40e8a94f228: Verifying Checksum
c40e8a94f228: Download complete
c2ed4d189f98: Verifying Checksum
c2ed4d189f98: Download complete
05cfa4dfe4e0: Verifying Checksum
05cfa4dfe4e0: Download complete
1e96c343b06a: Verifying Checksum
f057ee848829: Download complete
d78d453fce70: Verifying Checksum
d78d453fce70: Download complete
38512380f3ae: Verifying Checksum
38512380f3ae: Download complete
c63732422ad3: Verifying Checksum
c63732422ad3: Download complete
7375eb43508f: Verifying Checksum
7375eb43508f: Download complete
69d13bb66a4e: Verifying Checksum
69d13bb66a4e: Download complete
5add3b44d39b: Pull complete
9b279b7a7520: Pull complete
a22d25e0a88f: Pull complete
54401ce2cd58: Pull complete
95283fb14fdd: Pull complete
a415f739b308: Pull complete
a9bb3fcb22c8: Pull complete
c63732422ad3: Pull complete
c7fa0aeea839: Pull complete
15773f9a9d11: Pull complete
b070e225a67d: Pull complete
a63805ca0e65: Pull complete
086a99966bf6: Pull complete
ba0755234ac7: Pull complete
c2ed4d189f98: Pull complete
c40e8a94f228: Pull complete
69d13bb66a4e: Pull complete
05cfa4dfe4e0: Pull complete
1e96c343b06a: Pull complete
f057ee848829: Pull complete
d78d453fce70: Pull complete
38512380f3ae: Pull complete
7375eb43508f: Pull complete
Digest: sha256:b3b47805231a305d0f40c4bf069df20f6a2635574e6d4259fac651d3f9f6e098
Status: Downloaded newer image for googleapis/artman:latest
2020-05-13 05:12:17,920 synthtool [DEBUG] > Cloning googleapis.
2020-05-13 05:12:19,634 synthtool [DEBUG] > Running generator for google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-05-13 05:12:49,935 synthtool [SUCCESS] > Generated code into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2beta1.
2020-05-13 05:12:49,936 synthtool [SUCCESS] > Placed proto files into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2beta1/google/cloud/dialogflow_v2beta1/proto.
2020-05-13 05:12:50,372 synthtool [WARNING] > No replacements made in ['docs/**/*.rst', 'docs/conf.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-13 05:12:50,374 synthtool [INFO] > Replaced 'google.cloud.dialogflow' in docs/gapic/v2beta1/api.rst.
2020-05-13 05:12:50,374 synthtool [INFO] > Replaced 'google.cloud.dialogflow' in docs/gapic/v2beta1/types.rst.
2020-05-13 05:12:50,383 synthtool [WARNING] > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2beta1/**/*.py'] for pattern import google.cloud.dialogflow, maybe replacement is no longer needed?
2020-05-13 05:12:50,384 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_session_entity_types_client_v2beta1.py.
2020-05-13 05:12:50,385 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_agents_client_v2beta1.py.
2020-05-13 05:12:50,385 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_environments_client_v2beta1.py.
2020-05-13 05:12:50,385 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_entity_types_client_v2beta1.py.
2020-05-13 05:12:50,386 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_intents_client_v2beta1.py.
2020-05-13 05:12:50,386 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_knowledge_bases_client_v2beta1.py.
2020-05-13 05:12:50,386 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_sessions_client_v2beta1.py.
2020-05-13 05:12:50,386 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_contexts_client_v2beta1.py.
2020-05-13 05:12:50,387 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_documents_client_v2beta1.py.
2020-05-13 05:12:50,387 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/types.py.
2020-05-13 05:12:50,387 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/__init__.py.
2020-05-13 05:12:50,387 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_entity_type_pb2_grpc.py.
2020-05-13 05:12:50,388 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_entity_type_pb2.py.
2020-05-13 05:12:50,388 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/environment_pb2.py.
2020-05-13 05:12:50,389 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/intent_pb2.py.
2020-05-13 05:12:50,389 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/intent_pb2_grpc.py.
2020-05-13 05:12:50,390 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/webhook_pb2.py.
2020-05-13 05:12:50,390 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/entity_type_pb2_grpc.py.
2020-05-13 05:12:50,390 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/document_pb2.py.
2020-05-13 05:12:50,391 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/context_pb2_grpc.py.
2020-05-13 05:12:50,391 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/agent_pb2_grpc.py.
2020-05-13 05:12:50,391 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/document_pb2_grpc.py.
2020-05-13 05:12:50,392 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/agent_pb2.py.
2020-05-13 05:12:50,392 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_pb2.py.
2020-05-13 05:12:50,393 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/knowledge_base_pb2_grpc.py.
2020-05-13 05:12:50,393 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_pb2_grpc.py.
2020-05-13 05:12:50,393 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/environment_pb2_grpc.py.
2020-05-13 05:12:50,394 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-13 05:12:50,394 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-13 05:12:50,394 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-13 05:12:50,395 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-13 05:12:50,395 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-13 05:12:50,396 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-13 05:12:50,396 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-13 05:12:50,396 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-13 05:12:50,397 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-13 05:12:50,397 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/knowledge_bases_grpc_transport.py.
2020-05-13 05:12:50,397 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/sessions_grpc_transport.py.
2020-05-13 05:12:50,398 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/environments_grpc_transport.py.
2020-05-13 05:12:50,398 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/session_entity_types_grpc_transport.py.
2020-05-13 05:12:50,398 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/contexts_grpc_transport.py.
2020-05-13 05:12:50,398 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/intents_grpc_transport.py.
2020-05-13 05:12:50,399 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/entity_types_grpc_transport.py.
2020-05-13 05:12:50,399 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/agents_grpc_transport.py.
2020-05-13 05:12:50,399 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/documents_grpc_transport.py.
2020-05-13 05:12:50,400 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_session_entity_types_client_v2beta1.py.
2020-05-13 05:12:50,401 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_agents_client_v2beta1.py.
2020-05-13 05:12:50,401 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_environments_client_v2beta1.py.
2020-05-13 05:12:50,401 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_entity_types_client_v2beta1.py.
2020-05-13 05:12:50,402 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_intents_client_v2beta1.py.
2020-05-13 05:12:50,402 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_knowledge_bases_client_v2beta1.py.
2020-05-13 05:12:50,402 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_sessions_client_v2beta1.py.
2020-05-13 05:12:50,402 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_contexts_client_v2beta1.py.
2020-05-13 05:12:50,402 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_documents_client_v2beta1.py.
2020-05-13 05:12:50,406 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-13 05:12:50,407 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-13 05:12:50,407 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-13 05:12:50,407 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-13 05:12:50,407 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-13 05:12:50,408 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-13 05:12:50,409 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-13 05:12:50,409 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-13 05:12:50,409 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-13 05:12:50,418 synthtool [WARNING] > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2beta1/**/*.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-13 05:12:50,423 synthtool [INFO] > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-13 05:12:50,424 synthtool [INFO] > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-13 05:12:50,424 synthtool [INFO] > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-13 05:12:50,424 synthtool [INFO] > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-13 05:12:50,425 synthtool [INFO] > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-13 05:12:50,425 synthtool [INFO] > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-13 05:12:50,426 synthtool [INFO] > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-13 05:12:50,426 synthtool [INFO] > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-13 05:12:50,427 synthtool [INFO] > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-13 05:12:50,433 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-13 05:12:50,434 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-13 05:12:50,434 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-13 05:12:50,434 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-13 05:12:50,435 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-13 05:12:50,435 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-13 05:12:50,436 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-13 05:12:50,436 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-13 05:12:50,437 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-13 05:12:50,442 synthtool [WARNING] > No replacements made in dialogflow_v2beta1/gapic/agents_client.py for pattern (\s+).. raw:: html\s+<pre>curl, maybe replacement is no longer needed?
2020-05-13 05:12:50,442 synthtool [DEBUG] > Running generator for google/cloud/dialogflow/v2/artman_dialogflow_v2.yaml.
2020-05-13 05:13:14,996 synthtool [SUCCESS] > Generated code into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2.
2020-05-13 05:13:14,996 synthtool [SUCCESS] > Placed proto files into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2/google/cloud/dialogflow_v2/proto.
2020-05-13 05:13:15,417 synthtool [WARNING] > No replacements made in ['docs/**/*.rst', 'docs/conf.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-13 05:13:15,418 synthtool [INFO] > Replaced 'google.cloud.dialogflow' in docs/gapic/v2/api.rst.
2020-05-13 05:13:15,418 synthtool [INFO] > Replaced 'google.cloud.dialogflow' in docs/gapic/v2/types.rst.
2020-05-13 05:13:15,426 synthtool [WARNING] > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2/**/*.py'] for pattern import google.cloud.dialogflow, maybe replacement is no longer needed?
2020-05-13 05:13:15,426 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_sessions_client_v2.py.
2020-05-13 05:13:15,426 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_entity_types_client_v2.py.
2020-05-13 05:13:15,427 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_contexts_client_v2.py.
2020-05-13 05:13:15,427 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_session_entity_types_client_v2.py.
2020-05-13 05:13:15,427 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_intents_client_v2.py.
2020-05-13 05:13:15,427 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_agents_client_v2.py.
2020-05-13 05:13:15,427 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_environments_client_v2.py.
2020-05-13 05:13:15,429 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/types.py.
2020-05-13 05:13:15,429 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/__init__.py.
2020-05-13 05:13:15,429 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_entity_type_pb2_grpc.py.
2020-05-13 05:13:15,429 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_entity_type_pb2.py.
2020-05-13 05:13:15,430 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/intent_pb2.py.
2020-05-13 05:13:15,431 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/intent_pb2_grpc.py.
2020-05-13 05:13:15,431 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/webhook_pb2.py.
2020-05-13 05:13:15,431 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/entity_type_pb2_grpc.py.
2020-05-13 05:13:15,431 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/context_pb2_grpc.py.
2020-05-13 05:13:15,432 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/agent_pb2_grpc.py.
2020-05-13 05:13:15,432 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/agent_pb2.py.
2020-05-13 05:13:15,433 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_pb2.py.
2020-05-13 05:13:15,433 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_pb2_grpc.py.
2020-05-13 05:13:15,433 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/environment_pb2_grpc.py.
2020-05-13 05:13:15,434 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/agents_client.py.
2020-05-13 05:13:15,434 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/intents_client.py.
2020-05-13 05:13:15,434 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-13 05:13:15,434 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-13 05:13:15,436 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/environments_client.py.
2020-05-13 05:13:15,436 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/sessions_client.py.
2020-05-13 05:13:15,437 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/contexts_client.py.
2020-05-13 05:13:15,437 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/sessions_grpc_transport.py.
2020-05-13 05:13:15,438 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/environments_grpc_transport.py.
2020-05-13 05:13:15,438 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/session_entity_types_grpc_transport.py.
2020-05-13 05:13:15,438 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/contexts_grpc_transport.py.
2020-05-13 05:13:15,438 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/intents_grpc_transport.py.
2020-05-13 05:13:15,438 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/entity_types_grpc_transport.py.
2020-05-13 05:13:15,438 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/agents_grpc_transport.py.
2020-05-13 05:13:15,439 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_sessions_client_v2.py.
2020-05-13 05:13:15,439 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_entity_types_client_v2.py.
2020-05-13 05:13:15,439 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_contexts_client_v2.py.
2020-05-13 05:13:15,440 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_session_entity_types_client_v2.py.
2020-05-13 05:13:15,440 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_intents_client_v2.py.
2020-05-13 05:13:15,440 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_agents_client_v2.py.
2020-05-13 05:13:15,440 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_environments_client_v2.py.
2020-05-13 05:13:15,444 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2/gapic/agents_client.py.
2020-05-13 05:13:15,444 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2/gapic/intents_client.py.
2020-05-13 05:13:15,445 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-13 05:13:15,445 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-13 05:13:15,446 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2/gapic/environments_client.py.
2020-05-13 05:13:15,446 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2/gapic/sessions_client.py.
2020-05-13 05:13:15,446 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2/gapic/contexts_client.py.
2020-05-13 05:13:15,452 synthtool [INFO] > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/agents_client.py.
2020-05-13 05:13:15,452 synthtool [INFO] > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/intents_client.py.
2020-05-13 05:13:15,452 synthtool [INFO] > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-13 05:13:15,453 synthtool [INFO] > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-13 05:13:15,453 synthtool [INFO] > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/environments_client.py.
2020-05-13 05:13:15,454 synthtool [INFO] > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/sessions_client.py.
2020-05-13 05:13:15,454 synthtool [INFO] > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/contexts_client.py.
2020-05-13 05:13:15,461 synthtool [WARNING] > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2/**/*.py'] for pattern '-dialogflow', maybe replacement is no longer needed?
2020-05-13 05:13:15,466 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/agents_client.py.
2020-05-13 05:13:15,467 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/intents_client.py.
2020-05-13 05:13:15,467 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-13 05:13:15,467 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-13 05:13:15,468 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/environments_client.py.
2020-05-13 05:13:15,468 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/sessions_client.py.
2020-05-13 05:13:15,468 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/contexts_client.py.
2020-05-13 05:13:15,473 synthtool [WARNING] > No replacements made in dialogflow_v2/gapic/agents_client.py for pattern (\s+).. raw:: html\s+<pre>curl, maybe replacement is no longer needed?
2020-05-13 05:13:15,474 synthtool [INFO] > Replaced '# Generated by the .*' in dialogflow_v2beta1/proto/session_pb2.py.
2020-05-13 05:13:15,474 synthtool [INFO] > Replaced '# Generated by the .*' in dialogflow_v2beta1/proto/intent_pb2_grpc.py.
2020-05-13 05:13:15,474 synthtool [INFO] > Replaced '# Generated by the .*' in dialogflow_v2/proto/intent_pb2_grpc.py.
2020-05-13 05:13:15,475 synthtool [INFO] > Replaced '# Generated by the .*' in dialogflow_v2/proto/session_pb2.py.
2020-05-13 05:13:15,476 synthtool [WARNING] > No replacements made in ['dialogflow_v2beta1/gapic/intents_client.py', 'dialogflow_v2beta1/gapic/sessions_client.py', 'dialogflow_v2/gapic/intents_client.py'] for pattern # Copyright 2018 Google LLC, maybe replacement is no longer needed?
2020-05-13 05:13:15,476 synthtool [WARNING] > No replacements made in dialogflow_v2/gapic/agents_client.py for pattern }\\\" [\\]\n(\s+retry \(Optional), maybe replacement is no longer needed?
2020-05-13 05:13:15,483 synthtool [WARNING] > No replacements made in dialogflow_v2beta1/proto/intent_pb2.py for pattern \s+-----------, maybe replacement is no longer needed?
2020-05-13 05:13:15,484 synthtool [WARNING] > No replacements made in dialogflow_*/proto/session_pb2.py for pattern ============================================================================, maybe replacement is no longer needed?
2020-05-13 05:13:15,486 synthtool [INFO] > Replaced '\\s*\\<\\>`__' in dialogflow_v2beta1/proto/audio_config_pb2.py.
2020-05-13 05:13:15,487 synthtool [INFO] > Replaced '\\s*\\<\\>`__' in dialogflow_v2/proto/audio_config_pb2.py.
2020-05-13 05:13:15,488 synthtool [WARNING] > No replacements made in dialogflow_v2/proto/agent_pb2.py for pattern :math:, maybe replacement is no longer needed?
2020-05-13 05:13:15,488 synthtool [WARNING] > No replacements made in dialogflow_v2/proto/agent_pb2.py for pattern :raw-latex:, maybe replacement is no longer needed?
Error while collecting sessions.
Sessions not found: blacken
2020-05-13 05:13:15,693 synthtool [ERROR] > Failed executing nox -s blacken:

None
2020-05-13 05:13:15,694 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py", line 122, in <module>
    s.shell.run(["nox", "-s", "blacken"], hide_output=False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['nox', '-s', 'blacken']' returned non-zero exit status 3.
2020-05-13 05:13:15,738 autosynth [ERROR] > Synthesis failed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 600, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 472, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 550, in _inner_main
    ).synthesize(base_synth_log_path)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](https://sponge/73467609-370f-441b-aa99-2c568dc8113e).

Autosynth is still having trouble generating dialogflow-python-client-v2. :sob:

Here's the output from running `synth.py`:

```
2020-05-14 05:11:38,762 autosynth [INFO] > logs will be written to: /tmpfs/src/github/synthtool/logs/googleapis/dialogflow-python-client-v2
2020-05-14 05:11:39,345 autosynth [DEBUG] > Running: git config --global core.excludesfile /home/kbuilder/.autosynth-gitignore
2020-05-14 05:11:39,348 autosynth [DEBUG] > Running: git config user.name yoshi-automation
2020-05-14 05:11:39,350 autosynth [DEBUG] > Running: git config user.email yoshi-automation@google.com
2020-05-14 05:11:39,353 autosynth [DEBUG] > Running: git config push.default simple
2020-05-14 05:11:39,355 autosynth [DEBUG] > Running: git branch -f autosynth
2020-05-14 05:11:39,358 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-05-14 05:11:39,590 autosynth [INFO] > Running synthtool
2020-05-14 05:11:39,591 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-14 05:11:39,595 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-05-14 05:11:39,806 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py.
On branch autosynth
nothing to commit, working tree clean
2020-05-14 05:11:39,962 synthtool [DEBUG] > Ensuring dependencies.
2020-05-14 05:11:39,967 synthtool [DEBUG] > Pulling artman image.
latest: Pulling from googleapis/artman
fe703b657a32: Pulling fs layer
f9df1fafd224: Pulling fs layer
a645a4b887f9: Pulling fs layer
57db7fe0b522: Pulling fs layer
ad7df9457e38: Pulling fs layer
5add3b44d39b: Pulling fs layer
9b279b7a7520: Pulling fs layer
a22d25e0a88f: Pulling fs layer
54401ce2cd58: Pulling fs layer
95283fb14fdd: Pulling fs layer
a415f739b308: Pulling fs layer
a9bb3fcb22c8: Pulling fs layer
c63732422ad3: Pulling fs layer
c7fa0aeea839: Pulling fs layer
15773f9a9d11: Pulling fs layer
b070e225a67d: Pulling fs layer
a63805ca0e65: Pulling fs layer
086a99966bf6: Pulling fs layer
ba0755234ac7: Pulling fs layer
c2ed4d189f98: Pulling fs layer
c40e8a94f228: Pulling fs layer
69d13bb66a4e: Pulling fs layer
05cfa4dfe4e0: Pulling fs layer
1e96c343b06a: Pulling fs layer
f057ee848829: Pulling fs layer
57db7fe0b522: Waiting
d78d453fce70: Pulling fs layer
38512380f3ae: Pulling fs layer
ad7df9457e38: Waiting
5add3b44d39b: Waiting
7375eb43508f: Pulling fs layer
9b279b7a7520: Waiting
a22d25e0a88f: Waiting
15773f9a9d11: Waiting
54401ce2cd58: Waiting
69d13bb66a4e: Waiting
95283fb14fdd: Waiting
05cfa4dfe4e0: Waiting
a415f739b308: Waiting
1e96c343b06a: Waiting
f057ee848829: Waiting
a9bb3fcb22c8: Waiting
d78d453fce70: Waiting
c63732422ad3: Waiting
c7fa0aeea839: Waiting
38512380f3ae: Waiting
7375eb43508f: Waiting
b070e225a67d: Waiting
c2ed4d189f98: Waiting
c40e8a94f228: Waiting
086a99966bf6: Waiting
ba0755234ac7: Waiting
a63805ca0e65: Waiting
f9df1fafd224: Download complete
a645a4b887f9: Verifying Checksum
a645a4b887f9: Download complete
57db7fe0b522: Verifying Checksum
57db7fe0b522: Download complete
fe703b657a32: Verifying Checksum
fe703b657a32: Download complete
ad7df9457e38: Verifying Checksum
ad7df9457e38: Download complete
9b279b7a7520: Verifying Checksum
9b279b7a7520: Download complete
a22d25e0a88f: Verifying Checksum
a22d25e0a88f: Download complete
95283fb14fdd: Verifying Checksum
95283fb14fdd: Download complete
54401ce2cd58: Verifying Checksum
54401ce2cd58: Download complete
a9bb3fcb22c8: Verifying Checksum
a9bb3fcb22c8: Download complete
a415f739b308: Download complete
c7fa0aeea839: Verifying Checksum
c7fa0aeea839: Download complete
5add3b44d39b: Verifying Checksum
5add3b44d39b: Download complete
fe703b657a32: Pull complete
f9df1fafd224: Pull complete
a645a4b887f9: Pull complete
57db7fe0b522: Pull complete
b070e225a67d: Verifying Checksum
b070e225a67d: Download complete
15773f9a9d11: Verifying Checksum
15773f9a9d11: Download complete
a63805ca0e65: Verifying Checksum
a63805ca0e65: Download complete
c63732422ad3: Verifying Checksum
c63732422ad3: Download complete
086a99966bf6: Verifying Checksum
086a99966bf6: Download complete
ba0755234ac7: Download complete
c40e8a94f228: Download complete
05cfa4dfe4e0: Verifying Checksum
05cfa4dfe4e0: Download complete
ad7df9457e38: Pull complete
1e96c343b06a: Verifying Checksum
1e96c343b06a: Download complete
f057ee848829: Verifying Checksum
f057ee848829: Download complete
c2ed4d189f98: Verifying Checksum
c2ed4d189f98: Download complete
d78d453fce70: Verifying Checksum
d78d453fce70: Download complete
38512380f3ae: Verifying Checksum
38512380f3ae: Download complete
7375eb43508f: Verifying Checksum
7375eb43508f: Download complete
69d13bb66a4e: Verifying Checksum
69d13bb66a4e: Download complete
5add3b44d39b: Pull complete
9b279b7a7520: Pull complete
a22d25e0a88f: Pull complete
54401ce2cd58: Pull complete
95283fb14fdd: Pull complete
a415f739b308: Pull complete
a9bb3fcb22c8: Pull complete
c63732422ad3: Pull complete
c7fa0aeea839: Pull complete
15773f9a9d11: Pull complete
b070e225a67d: Pull complete
a63805ca0e65: Pull complete
086a99966bf6: Pull complete
ba0755234ac7: Pull complete
c2ed4d189f98: Pull complete
c40e8a94f228: Pull complete
69d13bb66a4e: Pull complete
05cfa4dfe4e0: Pull complete
1e96c343b06a: Pull complete
f057ee848829: Pull complete
d78d453fce70: Pull complete
38512380f3ae: Pull complete
7375eb43508f: Pull complete
Digest: sha256:b3b47805231a305d0f40c4bf069df20f6a2635574e6d4259fac651d3f9f6e098
Status: Downloaded newer image for googleapis/artman:latest
2020-05-14 05:12:17,967 synthtool [DEBUG] > Cloning googleapis.
2020-05-14 05:12:19,656 synthtool [DEBUG] > Running generator for google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-05-14 05:12:48,556 synthtool [SUCCESS] > Generated code into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2beta1.
2020-05-14 05:12:48,557 synthtool [SUCCESS] > Placed proto files into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2beta1/google/cloud/dialogflow_v2beta1/proto.
2020-05-14 05:12:49,022 synthtool [WARNING] > No replacements made in ['docs/**/*.rst', 'docs/conf.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-14 05:12:49,024 synthtool [INFO] > Replaced 'google.cloud.dialogflow' in docs/gapic/v2beta1/api.rst.
2020-05-14 05:12:49,024 synthtool [INFO] > Replaced 'google.cloud.dialogflow' in docs/gapic/v2beta1/types.rst.
2020-05-14 05:12:49,035 synthtool [WARNING] > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2beta1/**/*.py'] for pattern import google.cloud.dialogflow, maybe replacement is no longer needed?
2020-05-14 05:12:49,036 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_session_entity_types_client_v2beta1.py.
2020-05-14 05:12:49,036 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_agents_client_v2beta1.py.
2020-05-14 05:12:49,037 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_environments_client_v2beta1.py.
2020-05-14 05:12:49,037 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_entity_types_client_v2beta1.py.
2020-05-14 05:12:49,037 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_intents_client_v2beta1.py.
2020-05-14 05:12:49,037 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_knowledge_bases_client_v2beta1.py.
2020-05-14 05:12:49,037 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_sessions_client_v2beta1.py.
2020-05-14 05:12:49,038 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_contexts_client_v2beta1.py.
2020-05-14 05:12:49,038 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2beta1/test_documents_client_v2beta1.py.
2020-05-14 05:12:49,038 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/types.py.
2020-05-14 05:12:49,039 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/__init__.py.
2020-05-14 05:12:49,039 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_entity_type_pb2_grpc.py.
2020-05-14 05:12:49,039 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_entity_type_pb2.py.
2020-05-14 05:12:49,040 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/environment_pb2.py.
2020-05-14 05:12:49,040 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/intent_pb2.py.
2020-05-14 05:12:49,041 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/intent_pb2_grpc.py.
2020-05-14 05:12:49,041 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/webhook_pb2.py.
2020-05-14 05:12:49,042 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/entity_type_pb2_grpc.py.
2020-05-14 05:12:49,042 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/document_pb2.py.
2020-05-14 05:12:49,042 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/context_pb2_grpc.py.
2020-05-14 05:12:49,043 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/agent_pb2_grpc.py.
2020-05-14 05:12:49,043 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/document_pb2_grpc.py.
2020-05-14 05:12:49,043 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/agent_pb2.py.
2020-05-14 05:12:49,044 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_pb2.py.
2020-05-14 05:12:49,044 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/knowledge_base_pb2_grpc.py.
2020-05-14 05:12:49,045 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/session_pb2_grpc.py.
2020-05-14 05:12:49,045 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/proto/environment_pb2_grpc.py.
2020-05-14 05:12:49,045 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-14 05:12:49,046 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-14 05:12:49,046 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-14 05:12:49,046 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-14 05:12:49,047 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-14 05:12:49,047 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-14 05:12:49,048 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-14 05:12:49,048 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-14 05:12:49,049 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-14 05:12:49,049 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/knowledge_bases_grpc_transport.py.
2020-05-14 05:12:49,049 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/sessions_grpc_transport.py.
2020-05-14 05:12:49,049 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/environments_grpc_transport.py.
2020-05-14 05:12:49,050 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/session_entity_types_grpc_transport.py.
2020-05-14 05:12:49,050 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/contexts_grpc_transport.py.
2020-05-14 05:12:49,050 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/intents_grpc_transport.py.
2020-05-14 05:12:49,050 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/entity_types_grpc_transport.py.
2020-05-14 05:12:49,051 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/agents_grpc_transport.py.
2020-05-14 05:12:49,051 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2beta1/gapic/transports/documents_grpc_transport.py.
2020-05-14 05:12:49,052 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_session_entity_types_client_v2beta1.py.
2020-05-14 05:12:49,053 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_agents_client_v2beta1.py.
2020-05-14 05:12:49,053 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_environments_client_v2beta1.py.
2020-05-14 05:12:49,053 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_entity_types_client_v2beta1.py.
2020-05-14 05:12:49,053 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_intents_client_v2beta1.py.
2020-05-14 05:12:49,054 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_knowledge_bases_client_v2beta1.py.
2020-05-14 05:12:49,054 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_sessions_client_v2beta1.py.
2020-05-14 05:12:49,054 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_contexts_client_v2beta1.py.
2020-05-14 05:12:49,054 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2beta1/test_documents_client_v2beta1.py.
2020-05-14 05:12:49,058 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-14 05:12:49,059 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-14 05:12:49,059 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-14 05:12:49,059 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-14 05:12:49,060 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-14 05:12:49,060 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-14 05:12:49,061 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-14 05:12:49,061 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-14 05:12:49,061 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-14 05:12:49,072 synthtool [WARNING] > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2beta1/**/*.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-14 05:12:49,078 synthtool [INFO] > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-14 05:12:49,078 synthtool [INFO] > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-14 05:12:49,079 synthtool [INFO] > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-14 05:12:49,079 synthtool [INFO] > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-14 05:12:49,079 synthtool [INFO] > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-14 05:12:49,080 synthtool [INFO] > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-14 05:12:49,080 synthtool [INFO] > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-14 05:12:49,081 synthtool [INFO] > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-14 05:12:49,081 synthtool [INFO] > Replaced "'-dialogflow'" in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-14 05:12:49,089 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/documents_client.py.
2020-05-14 05:12:49,089 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/agents_client.py.
2020-05-14 05:12:49,090 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/intents_client.py.
2020-05-14 05:12:49,090 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/session_entity_types_client.py.
2020-05-14 05:12:49,090 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/entity_types_client.py.
2020-05-14 05:12:49,091 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/environments_client.py.
2020-05-14 05:12:49,092 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/sessions_client.py.
2020-05-14 05:12:49,092 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/contexts_client.py.
2020-05-14 05:12:49,092 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2beta1/gapic/knowledge_bases_client.py.
2020-05-14 05:12:49,097 synthtool [WARNING] > No replacements made in dialogflow_v2beta1/gapic/agents_client.py for pattern (\s+).. raw:: html\s+<pre>curl, maybe replacement is no longer needed?
2020-05-14 05:12:49,097 synthtool [DEBUG] > Running generator for google/cloud/dialogflow/v2/artman_dialogflow_v2.yaml.
2020-05-14 05:13:11,788 synthtool [SUCCESS] > Generated code into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2.
2020-05-14 05:13:11,789 synthtool [SUCCESS] > Placed proto files into /home/kbuilder/.cache/synthtool/googleapis/artman-genfiles/python/dialogflow-v2/google/cloud/dialogflow_v2/proto.
2020-05-14 05:13:12,232 synthtool [WARNING] > No replacements made in ['docs/**/*.rst', 'docs/conf.py'] for pattern google-cloud-dialogflow, maybe replacement is no longer needed?
2020-05-14 05:13:12,233 synthtool [INFO] > Replaced 'google.cloud.dialogflow' in docs/gapic/v2/api.rst.
2020-05-14 05:13:12,233 synthtool [INFO] > Replaced 'google.cloud.dialogflow' in docs/gapic/v2/types.rst.
2020-05-14 05:13:12,243 synthtool [WARNING] > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2/**/*.py'] for pattern import google.cloud.dialogflow, maybe replacement is no longer needed?
2020-05-14 05:13:12,243 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_sessions_client_v2.py.
2020-05-14 05:13:12,244 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_entity_types_client_v2.py.
2020-05-14 05:13:12,244 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_contexts_client_v2.py.
2020-05-14 05:13:12,244 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_session_entity_types_client_v2.py.
2020-05-14 05:13:12,244 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_intents_client_v2.py.
2020-05-14 05:13:12,245 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_agents_client_v2.py.
2020-05-14 05:13:12,245 synthtool [INFO] > Replaced 'from google.cloud\\.' in tests/unit/gapic/v2/test_environments_client_v2.py.
2020-05-14 05:13:12,246 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/types.py.
2020-05-14 05:13:12,246 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/__init__.py.
2020-05-14 05:13:12,247 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_entity_type_pb2_grpc.py.
2020-05-14 05:13:12,247 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_entity_type_pb2.py.
2020-05-14 05:13:12,248 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/intent_pb2.py.
2020-05-14 05:13:12,248 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/intent_pb2_grpc.py.
2020-05-14 05:13:12,249 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/webhook_pb2.py.
2020-05-14 05:13:12,249 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/entity_type_pb2_grpc.py.
2020-05-14 05:13:12,249 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/context_pb2_grpc.py.
2020-05-14 05:13:12,250 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/agent_pb2_grpc.py.
2020-05-14 05:13:12,250 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/agent_pb2.py.
2020-05-14 05:13:12,251 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_pb2.py.
2020-05-14 05:13:12,251 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/session_pb2_grpc.py.
2020-05-14 05:13:12,251 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/proto/environment_pb2_grpc.py.
2020-05-14 05:13:12,252 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/agents_client.py.
2020-05-14 05:13:12,252 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/intents_client.py.
2020-05-14 05:13:12,253 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-14 05:13:12,253 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-14 05:13:12,254 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/environments_client.py.
2020-05-14 05:13:12,255 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/sessions_client.py.
2020-05-14 05:13:12,255 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/contexts_client.py.
2020-05-14 05:13:12,256 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/sessions_grpc_transport.py.
2020-05-14 05:13:12,256 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/environments_grpc_transport.py.
2020-05-14 05:13:12,256 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/session_entity_types_grpc_transport.py.
2020-05-14 05:13:12,257 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/contexts_grpc_transport.py.
2020-05-14 05:13:12,257 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/intents_grpc_transport.py.
2020-05-14 05:13:12,257 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/entity_types_grpc_transport.py.
2020-05-14 05:13:12,257 synthtool [INFO] > Replaced 'from google.cloud\\.' in dialogflow_v2/gapic/transports/agents_grpc_transport.py.
2020-05-14 05:13:12,258 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_sessions_client_v2.py.
2020-05-14 05:13:12,258 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_entity_types_client_v2.py.
2020-05-14 05:13:12,258 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_contexts_client_v2.py.
2020-05-14 05:13:12,259 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_session_entity_types_client_v2.py.
2020-05-14 05:13:12,259 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_intents_client_v2.py.
2020-05-14 05:13:12,259 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_agents_client_v2.py.
2020-05-14 05:13:12,260 synthtool [INFO] > Replaced 'from google.cloud import' in tests/unit/gapic/v2/test_environments_client_v2.py.
2020-05-14 05:13:12,264 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2/gapic/agents_client.py.
2020-05-14 05:13:12,265 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2/gapic/intents_client.py.
2020-05-14 05:13:12,265 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-14 05:13:12,265 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-14 05:13:12,266 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2/gapic/environments_client.py.
2020-05-14 05:13:12,266 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2/gapic/sessions_client.py.
2020-05-14 05:13:12,267 synthtool [INFO] > Replaced 'from google.cloud import' in dialogflow_v2/gapic/contexts_client.py.
2020-05-14 05:13:12,273 synthtool [INFO] > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/agents_client.py.
2020-05-14 05:13:12,274 synthtool [INFO] > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/intents_client.py.
2020-05-14 05:13:12,274 synthtool [INFO] > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-14 05:13:12,274 synthtool [INFO] > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-14 05:13:12,275 synthtool [INFO] > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/environments_client.py.
2020-05-14 05:13:12,275 synthtool [INFO] > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/sessions_client.py.
2020-05-14 05:13:12,276 synthtool [INFO] > Replaced 'google-cloud-dialogflow' in dialogflow_v2/gapic/contexts_client.py.
2020-05-14 05:13:12,285 synthtool [WARNING] > No replacements made in ['tests/unit/gapic/**/*.py', 'dialogflow_v2/**/*.py'] for pattern '-dialogflow', maybe replacement is no longer needed?
2020-05-14 05:13:12,291 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/agents_client.py.
2020-05-14 05:13:12,291 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/intents_client.py.
2020-05-14 05:13:12,291 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/session_entity_types_client.py.
2020-05-14 05:13:12,292 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/entity_types_client.py.
2020-05-14 05:13:12,292 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/environments_client.py.
2020-05-14 05:13:12,293 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/sessions_client.py.
2020-05-14 05:13:12,293 synthtool [INFO] > Replaced '(Returns:\n\\s+)([a-zA-Z]+Client:)' in dialogflow_v2/gapic/contexts_client.py.
2020-05-14 05:13:12,298 synthtool [WARNING] > No replacements made in dialogflow_v2/gapic/agents_client.py for pattern (\s+).. raw:: html\s+<pre>curl, maybe replacement is no longer needed?
2020-05-14 05:13:12,299 synthtool [INFO] > Replaced '# Generated by the .*' in dialogflow_v2beta1/proto/session_pb2.py.
2020-05-14 05:13:12,299 synthtool [INFO] > Replaced '# Generated by the .*' in dialogflow_v2beta1/proto/intent_pb2_grpc.py.
2020-05-14 05:13:12,299 synthtool [INFO] > Replaced '# Generated by the .*' in dialogflow_v2/proto/intent_pb2_grpc.py.
2020-05-14 05:13:12,300 synthtool [INFO] > Replaced '# Generated by the .*' in dialogflow_v2/proto/session_pb2.py.
2020-05-14 05:13:12,301 synthtool [WARNING] > No replacements made in ['dialogflow_v2beta1/gapic/intents_client.py', 'dialogflow_v2beta1/gapic/sessions_client.py', 'dialogflow_v2/gapic/intents_client.py'] for pattern # Copyright 2018 Google LLC, maybe replacement is no longer needed?
2020-05-14 05:13:12,301 synthtool [WARNING] > No replacements made in dialogflow_v2/gapic/agents_client.py for pattern }\\\" [\\]\n(\s+retry \(Optional), maybe replacement is no longer needed?
2020-05-14 05:13:12,308 synthtool [WARNING] > No replacements made in dialogflow_v2beta1/proto/intent_pb2.py for pattern \s+-----------, maybe replacement is no longer needed?
2020-05-14 05:13:12,309 synthtool [WARNING] > No replacements made in dialogflow_*/proto/session_pb2.py for pattern ============================================================================, maybe replacement is no longer needed?
2020-05-14 05:13:12,310 synthtool [INFO] > Replaced '\\s*\\<\\>`__' in dialogflow_v2beta1/proto/audio_config_pb2.py.
2020-05-14 05:13:12,312 synthtool [INFO] > Replaced '\\s*\\<\\>`__' in dialogflow_v2/proto/audio_config_pb2.py.
2020-05-14 05:13:12,312 synthtool [WARNING] > No replacements made in dialogflow_v2/proto/agent_pb2.py for pattern :math:, maybe replacement is no longer needed?
2020-05-14 05:13:12,313 synthtool [WARNING] > No replacements made in dialogflow_v2/proto/agent_pb2.py for pattern :raw-latex:, maybe replacement is no longer needed?
Error while collecting sessions.
Sessions not found: blacken
2020-05-14 05:13:12,520 synthtool [ERROR] > Failed executing nox -s blacken:

None
2020-05-14 05:13:12,521 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py", line 122, in <module>
    s.shell.run(["nox", "-s", "blacken"], hide_output=False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['nox', '-s', 'blacken']' returned non-zero exit status 3.
2020-05-14 05:13:12,555 autosynth [ERROR] > Synthesis failed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 600, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 472, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 550, in _inner_main
    ).synthesize(base_synth_log_path)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](https://sponge/0993da46-cb2f-4f2d-bff1-f32543ab6b09).

--- a/synth.py
+++ b/synth.py
@@ -33,7 +33,7 @@ for version in versions:
             "README.rst",
             "docs/index.rst",
             "docs/conf.py",
-            "nox.py",
+            "noxfile.py",
         ],
     )
     s.move(library / f"google/cloud/dialogflow_{version}", f"dialogflow_{version}")

